--- a/net/minecraft/world/level/levelgen/presets/WorldPreset.java
+++ b/net/minecraft/world/level/levelgen/presets/WorldPreset.java
@@ -16,7 +_,7 @@
 import net.minecraft.world.level.levelgen.WorldDimensions;
 
 public class WorldPreset {
-   public static final Codec<WorldPreset> DIRECT_CODEC = RecordCodecBuilder.create(
+   public static final Codec<WorldPreset> DIRECT_CODEC = RecordCodecBuilder.<WorldPreset>create(
          i -> i.group(Codec.unboundedMap(ResourceKey.codec(Registries.LEVEL_STEM), LevelStem.CODEC).fieldOf("dimensions").forGetter(e -> e.dimensions))
             .apply(i, WorldPreset::new)
       )
