From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/commands/arguments/blocks/BlockStateParser.java b/net/minecraft/commands/arguments/blocks/BlockStateParser.java
index 3369affaf729d926dd2952f6c3d664aaa3089df4..8ab2dbb25933fdcdb0d068f0f80eaaae56c7d1b1 100644
--- a/net/minecraft/commands/arguments/blocks/BlockStateParser.java
+++ b/net/minecraft/commands/arguments/blocks/BlockStateParser.java
@@ -490,8 +490,8 @@ public class BlockStateParser {
    private <T extends Comparable<T>> void setValue(final Property<T> property, final String raw, final int start) throws CommandSyntaxException {
       Optional<T> value = property.getValue(raw);
       if (value.isPresent()) {
-         this.state = this.state.setValue(property, (Comparable)value.get());
-         this.properties.put(property, (Comparable)value.get());
+         this.state = this.state.setValue(property, value.get());
+         this.properties.put(property, value.get());
       } else {
          this.reader.setCursor(start);
          throw ERROR_INVALID_VALUE.createWithContext(this.reader, this.id.toString(), property.getName(), raw);
