--- a/net/minecraft/util/random/Weighted.java
+++ b/net/minecraft/util/random/Weighted.java
@@ -18,7 +_,7 @@
 
    public Weighted(T value, int weight) {
       if (weight < 0) {
-         throw (IllegalArgumentException)Util.pauseInIde((T)(new IllegalArgumentException("Weight should be >= 0")));
+         throw (IllegalArgumentException)Util.pauseInIde(new IllegalArgumentException("Weight should be >= 0"));
       } else {
          if (weight == 0 && SharedConstants.IS_RUNNING_IN_IDE) {
             LOGGER.warn("Found 0 weight, make sure this is intentional!");
