From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/core/registries/BuiltInRegistries.java b/net/minecraft/core/registries/BuiltInRegistries.java
index 6e28adee48600325a2600c93ebe618a847b6c3ab..ea20a8060ceef7fdec2a8ef2eb83819c8d3c07ae 100644
--- a/net/minecraft/core/registries/BuiltInRegistries.java
+++ b/net/minecraft/core/registries/BuiltInRegistries.java
@@ -387,8 +387,8 @@ public class BuiltInRegistries {
    ) {
       Bootstrap.checkBootstrapCalled(() -> "registry " + name.identifier());
       Identifier key = name.identifier();
-      LOADERS.put(key, (Supplier)() -> loader.run(registry));
-      WRITABLE_REGISTRY.register((ResourceKey<WritableRegistry<?>>)name, registry, RegistrationInfo.BUILT_IN);
+      LOADERS.put(key, () -> loader.run(registry));
+      WRITABLE_REGISTRY.register((ResourceKey) name, registry, RegistrationInfo.BUILT_IN);
       return registry;
    }
 
