From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/chunk/PalettedContainer.java b/net/minecraft/world/level/chunk/PalettedContainer.java
index e04070be787ac4b9bbda6ba789bb59604ad23a74..2be666ec71c207582a11ae7010cdabd63abccde1 100644
--- a/net/minecraft/world/level/chunk/PalettedContainer.java
+++ b/net/minecraft/world/level/chunk/PalettedContainer.java
@@ -49,17 +49,19 @@ public class PalettedContainer<T> implements PaletteResize<T>, PalettedContainer
    private static <T, C extends PalettedContainerRO<T>> Codec<C> codec(
       final Codec<T> elementCodec, final Strategy<T> strategy, final T defaultValue, final PalettedContainerRO.Unpacker<T, C> unpacker
    ) {
-      return RecordCodecBuilder.create(
-            i -> i.group(
-                  elementCodec.mapResult(ExtraCodecs.orElsePartial(defaultValue))
-                     .listOf()
-                     .fieldOf("palette")
-                     .forGetter(PalettedContainerRO.PackedData::paletteEntries),
-                  Codec.LONG_STREAM.lenientOptionalFieldOf("data").forGetter(PalettedContainerRO.PackedData::storage)
-               )
-               .apply(i, PalettedContainerRO.PackedData::new)
-         )
-         .comapFlatMap(discData -> unpacker.read(strategy, discData), palettedContainer -> palettedContainer.pack(strategy));
+      return RecordCodecBuilder.<PalettedContainerRO.PackedData>create(
+                      i -> i.group(
+                                      elementCodec.mapResult(ExtraCodecs.orElsePartial(defaultValue))
+                                              .listOf()
+                                              .fieldOf("palette")
+                                              .forGetter(PalettedContainerRO.PackedData::paletteEntries),
+                                      Codec.LONG_STREAM.lenientOptionalFieldOf("data").forGetter(PalettedContainerRO.PackedData::storage)
+                              )
+                              .apply(i, PalettedContainerRO.PackedData::new)
+              )
+              .comapFlatMap(
+                      discData -> unpacker.read(strategy, (PalettedContainerRO.PackedData<T>)discData), palettedContainer -> palettedContainer.pack(strategy)
+              );
    }
 
    private PalettedContainer(final Strategy<T> strategy, final Configuration dataConfiguration, final BitStorage storage, final Palette<T> palette) {
