From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/variant/PriorityProvider.java b/net/minecraft/world/entity/variant/PriorityProvider.java
index 785794fe68da6459e9cbe39554de124d8b6209cd..4f013f05495eeeefd0138a57249dea2e7bc00938 100644
--- a/net/minecraft/world/entity/variant/PriorityProvider.java
+++ b/net/minecraft/world/entity/variant/PriorityProvider.java
@@ -101,7 +101,7 @@ public interface PriorityProvider<Context, Condition extends PriorityProvider.Se
    }
 
    public record UnpackedEntry<C, T>(T entry, int priority, PriorityProvider.SelectorCondition<C> condition) {
-      public static final Comparator<PriorityProvider.UnpackedEntry<?, ?>> HIGHEST_PRIORITY_FIRST = Comparator.comparingInt(
+      public static final Comparator<PriorityProvider.UnpackedEntry<?, ?>> HIGHEST_PRIORITY_FIRST = Comparator.<PriorityProvider.UnpackedEntry<?, ?>>comparingInt(
             PriorityProvider.UnpackedEntry::priority
          )
          .reversed();
