From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/pathfinder/PathFinder.java b/net/minecraft/world/level/pathfinder/PathFinder.java
index 1f2fb45f258b8e3b320ac183753fa1969b9e9a70..f9cd36a980aef21978bb7f518f04655a8614c9cd 100644
--- a/net/minecraft/world/level/pathfinder/PathFinder.java
+++ b/net/minecraft/world/level/pathfinder/PathFinder.java
@@ -75,7 +75,7 @@ public class PathFinder {
       this.openSet.clear();
       this.openSet.insert(from);
       boolean captureDebug = this.captureDebug.getAsBoolean();
-      Set<Node> closedSet = (Set<Node>)(captureDebug ? new HashSet() : Set.of());
+      Set<Node> closedSet = (captureDebug ? new HashSet() : Set.of());
       int count = 0;
       Set<Target> reachedTargets = Sets.newHashSetWithExpectedSize(targets.size());
       int maxVisitedNodesAdjusted = (int)(this.maxVisitedNodes * maxVisitedNodesMultiplier);
