--- a/net/minecraft/server/commands/ReturnCommand.java
+++ b/net/minecraft/server/commands/ReturnCommand.java
@@ -19,11 +_,14 @@
 public class ReturnCommand {
    public static <T extends ExecutionCommandSource<T>> void register(final CommandDispatcher<T> dispatcher) {
       dispatcher.register(
-         (LiteralArgumentBuilder)((LiteralArgumentBuilder)((LiteralArgumentBuilder)((LiteralArgumentBuilder)LiteralArgumentBuilder.literal("return")
-                     .requires(Commands.hasPermission(Commands.LEVEL_GAMEMASTERS)))
-                  .then(RequiredArgumentBuilder.argument("value", IntegerArgumentType.integer()).executes(new ReturnCommand.ReturnValueCustomExecutor())))
-               .then(LiteralArgumentBuilder.literal("fail").executes(new ReturnCommand.ReturnFailCustomExecutor())))
-            .then(LiteralArgumentBuilder.literal("run").forward(dispatcher.getRoot(), new ReturnCommand.ReturnFromCommandCustomModifier(), false))
+              LiteralArgumentBuilder.<T>literal("return")
+                      .requires(Commands.hasPermission(Commands.LEVEL_GAMEMASTERS))
+                      .then(
+                              RequiredArgumentBuilder.<T, Integer>argument("value", IntegerArgumentType.integer())
+                                      .executes(new ReturnCommand.ReturnValueCustomExecutor<>())
+                      )
+                      .then(LiteralArgumentBuilder.<T>literal("fail").executes(new ReturnCommand.ReturnFailCustomExecutor<>()))
+                      .then(LiteralArgumentBuilder.<T>literal("run").forward(dispatcher.getRoot(), new ReturnCommand.ReturnFromCommandCustomModifier<>(), false))
       );
    }
 
