From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/random/Weighted.java b/net/minecraft/util/random/Weighted.java
index bf02e562d04dee075823cb0cd16d3a03e487c473..e5978dfc473a7ad1785eb99332aff65013688ee8 100644
--- a/net/minecraft/util/random/Weighted.java
+++ b/net/minecraft/util/random/Weighted.java
@@ -18,7 +18,7 @@ public record Weighted<T>(T value, int weight) {
 
    public Weighted(T value, int weight) {
       if (weight < 0) {
-         throw (IllegalArgumentException)Util.pauseInIde((T)(new IllegalArgumentException("Weight should be >= 0")));
+         throw (IllegalArgumentException)Util.pauseInIde(new IllegalArgumentException("Weight should be >= 0"));
       } else {
          if (weight == 0 && SharedConstants.IS_RUNNING_IN_IDE) {
             LOGGER.warn("Found 0 weight, make sure this is intentional!");
