From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/random/WeightedRandom.java b/net/minecraft/util/random/WeightedRandom.java
index e60bf29e977a31288aea2d81dddf94a8a902c8eb..1fb6845180faa5432c1e9a6dac4e91560ba9480a 100644
--- a/net/minecraft/util/random/WeightedRandom.java
+++ b/net/minecraft/util/random/WeightedRandom.java
@@ -26,7 +26,7 @@ public class WeightedRandom {
 
    public static <T> Optional<T> getRandomItem(final RandomSource random, final List<T> items, final int totalWeight, final ToIntFunction<T> weightGetter) {
       if (totalWeight < 0) {
-         throw (IllegalArgumentException)Util.pauseInIde((T)(new IllegalArgumentException("Negative total weight in getRandomItem")));
+         throw (IllegalArgumentException)Util.pauseInIde((new IllegalArgumentException("Negative total weight in getRandomItem")));
       } else if (totalWeight == 0) {
          return Optional.empty();
       } else {
