From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/schemas/V99.java b/net/minecraft/util/datafix/schemas/V99.java
index e11389e0e466cdc711cffa9987043d053742c806..23400211fb11a88d107bd7c9f51b3c6ae40a64f4 100644
--- a/net/minecraft/util/datafix/schemas/V99.java
+++ b/net/minecraft/util/datafix/schemas/V99.java
@@ -17,7 +17,7 @@ import org.slf4j.Logger;
 
 public class V99 extends Schema {
    private static final Logger LOGGER = LogUtils.getLogger();
-   private static final Map<String, String> ITEM_TO_BLOCKENTITY = (Map<String, String>)DataFixUtils.make(Maps.newHashMap(), map -> {
+   private static final Map<String, String> ITEM_TO_BLOCKENTITY = DataFixUtils.make(Maps.newHashMap(), map -> {
       map.put("minecraft:furnace", "Furnace");
       map.put("minecraft:lit_furnace", "Furnace");
       map.put("minecraft:chest", "Chest");
@@ -56,7 +56,7 @@ public class V99 extends Schema {
    public static final Map<String, String> ITEM_TO_ENTITY = Map.of("minecraft:armor_stand", "ArmorStand", "minecraft:painting", "Painting");
    protected static final HookFunction ADD_NAMES = new HookFunction() {
       public <T> T apply(final DynamicOps<T> ops, final T value) {
-         return V99.addNames(new Dynamic(ops, value), V99.ITEM_TO_BLOCKENTITY, V99.ITEM_TO_ENTITY);
+         return V99.addNames(new Dynamic<>(ops, value), V99.ITEM_TO_BLOCKENTITY, V99.ITEM_TO_ENTITY);
       }
    };
 
