From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/TicketStorage.java b/net/minecraft/world/level/TicketStorage.java
index 9a1fa183c1c1caca9916f9cb1be67e1ed41bc64e..40360e13eab46695b9d86ac5c21573510f385fae 100644
--- a/net/minecraft/world/level/TicketStorage.java
+++ b/net/minecraft/world/level/TicketStorage.java
@@ -90,7 +90,7 @@ public class TicketStorage extends SavedData {
          Entry<List<Ticket>> entry = (Entry<List<Ticket>>)var2.next();
          ChunkPos chunkPos = ChunkPos.unpack(entry.getLongKey());
 
-         for (Ticket ticket : (List)entry.getValue()) {
+         for (Ticket ticket : entry.getValue()) {
             output.accept(chunkPos, ticket);
          }
       }
@@ -102,7 +102,7 @@ public class TicketStorage extends SavedData {
       while (var1.hasNext()) {
          Entry<List<Ticket>> entry = (Entry<List<Ticket>>)var1.next();
 
-         for (Ticket ticket : (List)entry.getValue()) {
+         for (Ticket ticket : entry.getValue()) {
             this.addTicket(entry.getLongKey(), ticket);
          }
       }
@@ -376,7 +376,7 @@ public class TicketStorage extends SavedData {
       while (var4.hasNext()) {
          Entry<List<Ticket>> entry = (Entry<List<Ticket>>)var4.next();
 
-         for (Ticket ticket : (List)entry.getValue()) {
+         for (Ticket ticket : entry.getValue()) {
             if (ticket.getType() == ticketType) {
                affectedTickets.add(Pair.of(ticket, entry.getLongKey()));
             }
@@ -408,7 +408,7 @@ public class TicketStorage extends SavedData {
       while (var3.hasNext()) {
          Entry<List<Ticket>> entry = (Entry<List<Ticket>>)var3.next();
 
-         for (Ticket ticket : (List)entry.getValue()) {
+         for (Ticket ticket : entry.getValue()) {
             if (ticketCheck.test(ticket)) {
                chunks.add(entry.getLongKey());
                break;
