From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/network/chat/contents/NbtContents.java b/net/minecraft/network/chat/contents/NbtContents.java
index 4946a4c975cbc6125a7f90e2e13d661b914b1a7d..209ffa4bde7fdc327889c7ef03eb39a2b8e63fe0 100644
--- a/net/minecraft/network/chat/contents/NbtContents.java
+++ b/net/minecraft/network/chat/contents/NbtContents.java
@@ -43,7 +43,7 @@ public record NbtContents(
          }
       }
    );
-   public static final MapCodec<NbtContents> MAP_CODEC = RecordCodecBuilder.mapCodec(
+   public static final MapCodec<NbtContents> MAP_CODEC = RecordCodecBuilder.<NbtContents>mapCodec(
          i -> i.group(
                NBT_PATH_CODEC.fieldOf("nbt").forGetter(NbtContents::nbtPath),
                Codec.BOOL.lenientOptionalFieldOf("interpret", false).forGetter(NbtContents::interpreting),
