--- a/net/minecraft/world/level/block/HopperBlock.java
+++ b/net/minecraft/world/level/block/HopperBlock.java
@@ -54,7 +_,7 @@
       this.registerDefaultState(this.stateDefinition.any().setValue(FACING, Direction.DOWN).setValue(ENABLED, true));
       VoxelShape inside = Block.column(12.0, 11.0, 16.0);
       this.shapes = this.makeShapes(inside);
-      this.interactionShapes = ImmutableMap.builderWithExpectedSize(5)
+      this.interactionShapes = ImmutableMap.<Direction, VoxelShape>builderWithExpectedSize(5)
          .putAll(Shapes.rotateHorizontal(Shapes.or(inside, Block.boxZ(4.0, 8.0, 10.0, 0.0, 4.0))))
          .put(Direction.DOWN, inside)
          .build();
