From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/schemas/V704.java b/net/minecraft/util/datafix/schemas/V704.java
index 0c2159abd6fa12f2b019584f938fe5b19a2cbdcb..e5b9ba8ebde62d41e2491832a3077bc813c51f5f 100644
--- a/net/minecraft/util/datafix/schemas/V704.java
+++ b/net/minecraft/util/datafix/schemas/V704.java
@@ -141,7 +141,7 @@ public class V704 extends Schema {
    });
    protected static final HookFunction ADD_NAMES = new HookFunction() {
       public <T> T apply(final DynamicOps<T> ops, final T value) {
-         return V99.addNames(new Dynamic(ops, value), V704.ITEM_TO_BLOCKENTITY, V99.ITEM_TO_ENTITY);
+         return V99.addNames(new Dynamic<>(ops, value), V704.ITEM_TO_BLOCKENTITY, V99.ITEM_TO_ENTITY);
       }
    };
 
@@ -158,7 +158,7 @@ public class V704 extends Schema {
    public Map<String, Supplier<TypeTemplate>> registerBlockEntities(final Schema schema) {
       Map<String, Supplier<TypeTemplate>> map = super.registerBlockEntities(schema);
       BlockEntityIdFix.ID_MAP
-         .forEach((oldId, newId) -> map.put(newId, (Supplier)Objects.requireNonNull((Supplier)map.remove(oldId), () -> "Didn't find " + oldId + " in schema")));
+         .forEach((oldId, newId) -> map.put(newId, Objects.requireNonNull(map.remove(oldId), () -> "Didn't find " + oldId + " in schema")));
       return map;
    }
 
