--- a/net/minecraft/util/datafix/schemas/V704.java
+++ b/net/minecraft/util/datafix/schemas/V704.java
@@ -141,7 +_,7 @@
    });
    protected static final HookFunction ADD_NAMES = new HookFunction() {
       public <T> T apply(final DynamicOps<T> ops, final T value) {
-         return V99.addNames(new Dynamic(ops, value), V704.ITEM_TO_BLOCKENTITY, V99.ITEM_TO_ENTITY);
+         return V99.addNames(new Dynamic<>(ops, value), V704.ITEM_TO_BLOCKENTITY, V99.ITEM_TO_ENTITY);
       }
    };
 
@@ -158,7 +_,7 @@
    public Map<String, Supplier<TypeTemplate>> registerBlockEntities(final Schema schema) {
       Map<String, Supplier<TypeTemplate>> map = super.registerBlockEntities(schema);
       BlockEntityIdFix.ID_MAP
-         .forEach((oldId, newId) -> map.put(newId, (Supplier)Objects.requireNonNull((Supplier)map.remove(oldId), () -> "Didn't find " + oldId + " in schema")));
+         .forEach((oldId, newId) -> map.put(newId, Objects.requireNonNull(map.remove(oldId), () -> "Didn't find " + oldId + " in schema")));
       return map;
    }
 
