From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/advancements/criterion/StatePropertiesPredicate.java b/net/minecraft/advancements/criterion/StatePropertiesPredicate.java
index 6e18be34816e9f44dc79d5ce19f9272a0f45afbc..490c516fcc10045edb3a4045c7649e31c7b6e6a3 100644
--- a/net/minecraft/advancements/criterion/StatePropertiesPredicate.java
+++ b/net/minecraft/advancements/criterion/StatePropertiesPredicate.java
@@ -106,7 +106,7 @@ public record StatePropertiesPredicate(List<StatePropertiesPredicate.PropertyMat
       public <T extends Comparable<T>> boolean match(final StateHolder<?, ?> state, final Property<T> property) {
          T actualValue = state.getValue(property);
          Optional<T> typedExpected = property.getValue(this.value);
-         return typedExpected.isPresent() && actualValue.compareTo((Comparable)typedExpected.get()) == 0;
+         return typedExpected.isPresent() && actualValue.compareTo(typedExpected.get()) == 0;
       }
    }
 
@@ -151,14 +151,14 @@ public record StatePropertiesPredicate(List<StatePropertiesPredicate.PropertyMat
          T value = state.getValue(property);
          if (this.minValue.isPresent()) {
             Optional<T> typedMinValue = property.getValue((String)this.minValue.get());
-            if (typedMinValue.isEmpty() || value.compareTo((Comparable)typedMinValue.get()) < 0) {
+            if (typedMinValue.isEmpty() || value.compareTo(typedMinValue.get()) < 0) {
                return false;
             }
          }
 
          if (this.maxValue.isPresent()) {
             Optional<T> typedMaxValue = property.getValue((String)this.maxValue.get());
-            if (typedMaxValue.isEmpty() || value.compareTo((Comparable)typedMaxValue.get()) > 0) {
+            if (typedMaxValue.isEmpty() || value.compareTo(typedMaxValue.get()) > 0) {
                return false;
             }
          }
