--- a/net/minecraft/commands/arguments/selector/EntitySelector.java
+++ b/net/minecraft/commands/arguments/selector/EntitySelector.java
@@ -253,15 +_,15 @@
          List<Predicate<Entity>> predicates = new ObjectArrayList(this.contextFreePredicates.size() + extraCount);
          predicates.addAll(this.contextFreePredicates);
          if (filterFeatures) {
-            predicates.add((Predicate)e -> e.getType().isEnabled(enabledFeatures));
+            predicates.add(e -> e.getType().isEnabled(enabledFeatures));
          }
 
          if (filterAabb) {
-            predicates.add((Predicate)e -> absoluteAabb.intersects(e.getBoundingBox()));
+            predicates.add(e -> absoluteAabb.intersects(e.getBoundingBox()));
          }
 
          if (filterRange) {
-            predicates.add((Predicate)e -> this.range.matchesSqr(e.distanceToSqr(pos)));
+            predicates.add(e -> this.range.matchesSqr(e.distanceToSqr(pos)));
          }
 
          completePredicates = predicates;
