From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/animal/frog/TadpoleAi.java b/net/minecraft/world/entity/animal/frog/TadpoleAi.java
index a1fd42d68001e6609f61a70ef9cbf0afd4feb935..0c96796ee0e15eace1d5f7fce59b873c509a257f 100644
--- a/net/minecraft/world/entity/animal/frog/TadpoleAi.java
+++ b/net/minecraft/world/entity/animal/frog/TadpoleAi.java
@@ -37,7 +37,7 @@ public class TadpoleAi {
          Activity.CORE,
          0,
          ImmutableList.of(
-            new AnimalPanic(2.0F), new LookAtTargetSink(45, 90), new MoveToTargetSink(), new CountDownCooldownTicks(MemoryModuleType.TEMPTATION_COOLDOWN_TICKS)
+            new AnimalPanic<>(2.0F), new LookAtTargetSink(45, 90), new MoveToTargetSink(), new CountDownCooldownTicks(MemoryModuleType.TEMPTATION_COOLDOWN_TICKS)
          )
       );
    }
