From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/levelgen/structure/pieces/PiecesContainer.java b/net/minecraft/world/level/levelgen/structure/pieces/PiecesContainer.java
index c0f7ad54fb0461f43903618d0ad67023b083dc07..e5ba498d23201a4577cfbb0d0ecfbd431eeac9d8 100644
--- a/net/minecraft/world/level/levelgen/structure/pieces/PiecesContainer.java
+++ b/net/minecraft/world/level/levelgen/structure/pieces/PiecesContainer.java
@@ -19,7 +19,7 @@ import org.slf4j.Logger;
 public record PiecesContainer(List<StructurePiece> pieces) {
    private static final Logger LOGGER = LogUtils.getLogger();
    private static final Identifier JIGSAW_RENAME = Identifier.withDefaultNamespace("jigsaw");
-   private static final Map<Identifier, Identifier> RENAMES = ImmutableMap.builder()
+   private static final Map<Identifier, Identifier> RENAMES = ImmutableMap.<Identifier, Identifier>builder()
       .put(Identifier.withDefaultNamespace("nvi"), JIGSAW_RENAME)
       .put(Identifier.withDefaultNamespace("pcp"), JIGSAW_RENAME)
       .put(Identifier.withDefaultNamespace("bastionremnant"), JIGSAW_RENAME)
