--- a/net/minecraft/world/level/levelgen/structure/pieces/PiecesContainer.java
+++ b/net/minecraft/world/level/levelgen/structure/pieces/PiecesContainer.java
@@ -19,7 +_,7 @@
 public record PiecesContainer(List<StructurePiece> pieces) {
    private static final Logger LOGGER = LogUtils.getLogger();
    private static final Identifier JIGSAW_RENAME = Identifier.withDefaultNamespace("jigsaw");
-   private static final Map<Identifier, Identifier> RENAMES = ImmutableMap.builder()
+   private static final Map<Identifier, Identifier> RENAMES = ImmutableMap.<Identifier, Identifier>builder()
       .put(Identifier.withDefaultNamespace("nvi"), JIGSAW_RENAME)
       .put(Identifier.withDefaultNamespace("pcp"), JIGSAW_RENAME)
       .put(Identifier.withDefaultNamespace("bastionremnant"), JIGSAW_RENAME)
