From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/commands/synchronization/SingletonArgumentInfo.java b/net/minecraft/commands/synchronization/SingletonArgumentInfo.java
index 2fd2eec31d693ed25d662f0bd37adbdea1499201..ef4786118b3ee7d7d9156f1cbe89cc517cca9334 100644
--- a/net/minecraft/commands/synchronization/SingletonArgumentInfo.java
+++ b/net/minecraft/commands/synchronization/SingletonArgumentInfo.java
@@ -16,7 +16,7 @@ public class SingletonArgumentInfo<A extends ArgumentType<?>> implements Argumen
    }
 
    public static <T extends ArgumentType<?>> SingletonArgumentInfo<T> contextFree(final Supplier<T> constructor) {
-      return new SingletonArgumentInfo<>(context -> (ArgumentType)constructor.get());
+      return new SingletonArgumentInfo<>(context -> constructor.get());
    }
 
    public static <T extends ArgumentType<?>> SingletonArgumentInfo<T> contextAware(final Function<CommandBuildContext, T> constructor) {
