--- a/net/minecraft/world/level/dimension/end/EnderDragonFight.java
+++ b/net/minecraft/world/level/dimension/end/EnderDragonFight.java
@@ -103,7 +_,7 @@
    private @Nullable DragonRespawnStage respawnStage;
    private int respawnTime;
    private List<EntityReference<EndCrystal>> respawnCrystals;
-   public static final Codec<EnderDragonFight> CODEC = RecordCodecBuilder.create(
+   public static final Codec<EnderDragonFight> CODEC = RecordCodecBuilder.<EnderDragonFight>create(
       i -> i.group(
             Codec.BOOL.fieldOf("needs_state_scanning").orElse(true).forGetter(fight -> fight.needsStateScanning),
             Codec.BOOL.fieldOf("dragon_killed").orElse(false).forGetter(fight -> fight.dragonKilled),
@@ -112,8 +_,8 @@
             Codec.INT.fieldOf("respawn_time").orElse(0).forGetter(fight -> fight.respawnTime),
             UUIDUtil.CODEC.lenientOptionalFieldOf("dragon_uuid").forGetter(fight -> Optional.ofNullable(fight.dragonUUID)),
             BlockPos.CODEC.lenientOptionalFieldOf("exit_portal_location").forGetter(fight -> Optional.ofNullable(fight.exitPortalLocation)),
-            Codec.list(Codec.INT).lenientOptionalFieldOf("gateways", new ArrayList()).forGetter(fight -> fight.gateways),
-            Codec.list(EntityReference.codec()).optionalFieldOf("respawn_crystals", List.of()).forGetter(fight -> fight.respawnCrystals)
+            Codec.list(Codec.INT).lenientOptionalFieldOf("gateways", new ArrayList<>()).forGetter(fight -> fight.gateways),
+            Codec.list(EntityReference.<EndCrystal>codec()).optionalFieldOf("respawn_crystals", List.of()).forGetter(fight -> fight.respawnCrystals)
          )
          .apply(i, EnderDragonFight::new)
    );
