--- a/net/minecraft/world/entity/ai/sensing/GolemSensor.java
+++ b/net/minecraft/world/entity/ai/sensing/GolemSensor.java
@@ -33,7 +_,7 @@
    public static void checkForNearbyGolem(final LivingEntity body) {
       Optional<List<LivingEntity>> livingEntitiesMemory = body.getBrain().getMemory(MemoryModuleType.NEAREST_LIVING_ENTITIES);
       if (!livingEntitiesMemory.isEmpty()) {
-         boolean golemPresent = ((List)livingEntitiesMemory.get()).stream().anyMatch(entity -> entity.is(EntityType.IRON_GOLEM));
+         boolean golemPresent = (livingEntitiesMemory.get()).stream().anyMatch(entity -> entity.is(EntityType.IRON_GOLEM));
          if (golemPresent) {
             golemDetected(body);
          }
