From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/sensing/MobSensor.java b/net/minecraft/world/entity/ai/sensing/MobSensor.java
index 417275b924267d9e32ad627dbe2085d3f053738f..40c85eec16fefb8da835a4e3d75d0828a90bbddf 100644
--- a/net/minecraft/world/entity/ai/sensing/MobSensor.java
+++ b/net/minecraft/world/entity/ai/sensing/MobSensor.java
@@ -46,7 +46,7 @@ public class MobSensor<T extends LivingEntity> extends Sensor<T> {
    public void checkForMobsNearby(final T body) {
       Optional<List<LivingEntity>> livingEntitiesMemory = body.getBrain().getMemory(MemoryModuleType.NEAREST_LIVING_ENTITIES);
       if (!livingEntitiesMemory.isEmpty()) {
-         boolean mobPresent = ((List)livingEntitiesMemory.get()).stream().anyMatch(entity -> this.mobTest.test(body, entity));
+         boolean mobPresent = (livingEntitiesMemory.get()).stream().anyMatch(entity -> this.mobTest.test(body, entity));
          if (mobPresent) {
             this.mobDetected(body);
          }
