From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/levelgen/blending/BlendingData.java b/net/minecraft/world/level/levelgen/blending/BlendingData.java
index 635591ba820dd23788fc3b3bc156e196e3480e4b..1ec54dc92cd5302288e5f3768a4f2c22bf07bb4d 100644
--- a/net/minecraft/world/level/levelgen/blending/BlendingData.java
+++ b/net/minecraft/world/level/levelgen/blending/BlendingData.java
@@ -400,7 +400,7 @@ public class BlendingData {
 
    public record Packed(int minSection, int maxSection, Optional<double[]> heights) {
       private static final Codec<double[]> DOUBLE_ARRAY_CODEC = Codec.DOUBLE.listOf().xmap(Doubles::toArray, Doubles::asList);
-      public static final Codec<BlendingData.Packed> CODEC = RecordCodecBuilder.create(
+      public static final Codec<BlendingData.Packed> CODEC = RecordCodecBuilder.<BlendingData.Packed>create(
             i -> i.group(
                   Codec.INT.fieldOf("min_section").forGetter(BlendingData.Packed::minSection),
                   Codec.INT.fieldOf("max_section").forGetter(BlendingData.Packed::maxSection),
