--- a/net/minecraft/world/entity/animal/armadillo/ArmadilloAi.java
+++ b/net/minecraft/world/entity/animal/armadillo/ArmadilloAi.java
@@ -64,7 +_,7 @@
          Activity.CORE,
          0,
          ImmutableList.of(
-            new Swim(0.8F),
+            new Swim<>(0.8F),
             new ArmadilloAi.ArmadilloPanic(2.0F),
             new LookAtTargetSink(45, 90),
             new MoveToTargetSink() {
