From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/server/level/ServerPlayer.java b/net/minecraft/server/level/ServerPlayer.java
index 28ec6ddd4113a7a73b82f8b59fc7d3754f7687d9..d665e2fd91864acfea7ccb0d88cd59b0e81311c4 100644
--- a/net/minecraft/server/level/ServerPlayer.java
+++ b/net/minecraft/server/level/ServerPlayer.java
@@ -281,12 +281,7 @@ public class ServerPlayer extends Player {
             .maximumSize(256L)
             .build(
                new CacheLoader<TypedDataComponent<?>, Integer>() {
-                  private final DynamicOps<HashCode> registryHashOps;
-
-                  {
-                     Objects.requireNonNull(<VAR_NAMELESS_ENCLOSURE>);
-                     this.registryHashOps = ServerPlayer.this.registryAccess().createSerializationContext(HashOps.CRC32C_INSTANCE);
-                  }
+                  private final DynamicOps<HashCode> registryHashOps = ServerPlayer.this.registryAccess().createSerializationContext(HashOps.CRC32C_INSTANCE);
 
                   public Integer load(final TypedDataComponent<?> component) {
                      return ((HashCode)component.encodeValue(this.registryHashOps)
