From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/block/HopperBlock.java b/net/minecraft/world/level/block/HopperBlock.java
index 8d7e04917b41ff8a5d9505b43943ee61ac61d6b9..901c1b16d1d3514305aa406eaf2f548fc6f659b6 100644
--- a/net/minecraft/world/level/block/HopperBlock.java
+++ b/net/minecraft/world/level/block/HopperBlock.java
@@ -54,7 +54,7 @@ public class HopperBlock extends BaseEntityBlock {
       this.registerDefaultState(this.stateDefinition.any().setValue(FACING, Direction.DOWN).setValue(ENABLED, true));
       VoxelShape inside = Block.column(12.0, 11.0, 16.0);
       this.shapes = this.makeShapes(inside);
-      this.interactionShapes = ImmutableMap.builderWithExpectedSize(5)
+      this.interactionShapes = ImmutableMap.<Direction, VoxelShape>builderWithExpectedSize(5)
          .putAll(Shapes.rotateHorizontal(Shapes.or(inside, Block.boxZ(4.0, 8.0, 10.0, 0.0, 4.0))))
          .put(Direction.DOWN, inside)
          .build();
