From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/timeline/Timeline.java b/net/minecraft/world/timeline/Timeline.java
index c9a8bae039e45a74af0c1a8322cbfe0095382694..043d2927942a88f6a14d14ab884cfaaf6799a38d 100644
--- a/net/minecraft/world/timeline/Timeline.java
+++ b/net/minecraft/world/timeline/Timeline.java
@@ -34,7 +34,7 @@ public class Timeline {
    private static final Codec<Map<EnvironmentAttribute<?>, AttributeTrack<?, ?>>> TRACKS_CODEC = Codec.dispatchedMap(
       EnvironmentAttributes.CODEC, Util.memoize(AttributeTrack::createCodec)
    );
-   public static final Codec<Timeline> DIRECT_CODEC = RecordCodecBuilder.create(
+   public static final Codec<Timeline> DIRECT_CODEC = RecordCodecBuilder.<Timeline>create(
          i -> i.group(
                WorldClock.CODEC.fieldOf("clock").forGetter(t -> t.clock),
                ExtraCodecs.POSITIVE_INT.optionalFieldOf("period_ticks").forGetter(t -> t.periodTicks),
