From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/commands/arguments/selector/options/EntitySelectorOptions.java b/net/minecraft/commands/arguments/selector/options/EntitySelectorOptions.java
index 7624e0f29310f11f1a9b3ed1fdade5a885a34f2d..dd84a49dc945bfbc1e7549f63b5928964036f68d 100644
--- a/net/minecraft/commands/arguments/selector/options/EntitySelectorOptions.java
+++ b/net/minecraft/commands/arguments/selector/options/EntitySelectorOptions.java
@@ -400,7 +400,7 @@ public class EntitySelectorOptions {
                      reader.expect('=');
                      reader.skipWhitespace();
                      boolean value = reader.readBoolean();
-                     progress.put(criterion, (Predicate)p -> p.isDone() == value);
+                     progress.put(criterion, p -> p.isDone() == value);
                      reader.skipWhitespace();
                      if (reader.canRead() && reader.peek() == ',') {
                         reader.skip();
@@ -410,7 +410,7 @@ public class EntitySelectorOptions {
                   reader.skipWhitespace();
                   reader.expect('}');
                   reader.skipWhitespace();
-                  expected.put(name, (Predicate)p -> {
+                  expected.put(name, p -> {
                      for (Entry<String, Predicate<CriterionProgress>> entry : progress.entrySet()) {
                         CriterionProgress criterionx = p.getCriterion((String)entry.getKey());
                         if (criterionx == null || !((Predicate)entry.getValue()).test(criterionx)) {
@@ -422,7 +422,7 @@ public class EntitySelectorOptions {
                   });
                } else {
                   boolean value = reader.readBoolean();
-                  expected.put(name, (Predicate)p -> p.isDone() == value);
+                  expected.put(name, p -> p.isDone() == value);
                }
 
                reader.skipWhitespace();
