From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/block/state/properties/Property.java b/net/minecraft/world/level/block/state/properties/Property.java
index 2194bf36779a8e1dec119166deed70c8d2e06a13..836ac0d96f7cfd19ae2db382e4fcfa2b60ccfb3c 100644
--- a/net/minecraft/world/level/block/state/properties/Property.java
+++ b/net/minecraft/world/level/block/state/properties/Property.java
@@ -16,7 +16,7 @@ public abstract class Property<T extends Comparable<T>> {
    private @Nullable Integer hashCode;
    private final Codec<T> codec = Codec.STRING
       .comapFlatMap(
-         namex -> (DataResult)this.getValue(namex)
+         namex -> this.getValue(namex)
             .map(DataResult::success)
             .orElseGet(() -> DataResult.error(() -> "Unable to read property: " + this + " with value: " + namex)),
          this::getName
