--- a/net/minecraft/network/Connection.java
+++ b/net/minecraft/network/Connection.java
@@ -280,7 +_,7 @@
          this.flushQueue();
          this.sendPacket(packet, listener, flush);
       } else {
-         this.pendingActions.add((Consumer)connection -> connection.sendPacket(packet, listener, flush));
+         this.pendingActions.add(connection -> connection.sendPacket(packet, listener, flush));
       }
    }
 
