From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/resources/ResourceKey.java b/net/minecraft/resources/ResourceKey.java
index 44f8c4d87840dac98402361b11d8e31fdddced71..3b072fe037d3def0108c8985069054b87f4ff1d0 100644
--- a/net/minecraft/resources/ResourceKey.java
+++ b/net/minecraft/resources/ResourceKey.java
@@ -48,7 +48,7 @@ public class ResourceKey<T> {
    }
 
    public <E> Optional<ResourceKey<E>> cast(final ResourceKey<? extends Registry<E>> registry) {
-      return this.isFor(registry) ? Optional.of(this) : Optional.empty();
+      return this.isFor(registry) ? Optional.of((ResourceKey<E>)this) : Optional.empty();
    }
 
    public Identifier identifier() {
