From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/server/packs/resources/ResourceMetadata.java b/net/minecraft/server/packs/resources/ResourceMetadata.java
index ad177e34473c300f29c70ed921fd592dcf932997..07247f657822b92eebbf98e94052ab2f8ba47a4e 100644
--- a/net/minecraft/server/packs/resources/ResourceMetadata.java
+++ b/net/minecraft/server/packs/resources/ResourceMetadata.java
@@ -74,7 +74,7 @@ public interface ResourceMetadata {
    }
 
    default List<MetadataSectionType.WithValue<?>> getTypedSections(final Collection<MetadataSectionType<?>> types) {
-      return (List<MetadataSectionType.WithValue<?>>)types.stream()
+      return types.stream()
          .map(this::getTypedSection)
          .flatMap(Optional::stream)
          .collect(Collectors.toUnmodifiableList());
@@ -89,7 +89,7 @@ public interface ResourceMetadata {
 
       @Override
       public <T> Optional<T> getSection(final MetadataSectionType<T> serializer) {
-         return Optional.ofNullable(this.values.get(serializer));
+         return Optional.ofNullable((T)this.values.get(serializer));
       }
    }
 }
