From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/storage/loot/ValidationContext.java b/net/minecraft/world/level/storage/loot/ValidationContext.java
index 52d4a337a070d336ac61bcf1df4c6ee3bf435167..70146a50b5f619413c6e110094723c8c5d61be4c 100644
--- a/net/minecraft/world/level/storage/loot/ValidationContext.java
+++ b/net/minecraft/world/level/storage/loot/ValidationContext.java
@@ -53,7 +53,7 @@ public class ValidationContext {
    }
 
    public ValidationContext enterElement(final ProblemReporter.PathElement subContext, final ResourceKey<?> element) {
-      Set<ResourceKey<?>> newVisitedElements = ImmutableSet.builder().addAll(this.visitedElements).add(element).build();
+      Set<ResourceKey<?>> newVisitedElements = ImmutableSet.<ResourceKey<?>>builder().addAll(this.visitedElements).add(element).build();
       return new ValidationContext(this.reporter.forChild(subContext), this.contextKeySet, this.resolver, newVisitedElements);
    }
 
