From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/behavior/GoAndGiveItemsToTarget.java b/net/minecraft/world/entity/ai/behavior/GoAndGiveItemsToTarget.java
index 5f224a429a8ea6773e1457ac511b1bcf6db80b86..f44bca9a8f905c843f1517b3a7ba60ac145a8f23 100644
--- a/net/minecraft/world/entity/ai/behavior/GoAndGiveItemsToTarget.java
+++ b/net/minecraft/world/entity/ai/behavior/GoAndGiveItemsToTarget.java
@@ -55,7 +55,7 @@ public class GoAndGiveItemsToTarget<E extends LivingEntity & InventoryCarrier> e
 
    @Override
    protected void start(final ServerLevel level, final E body, final long timestamp) {
-      ((Optional)this.targetPositionGetter.apply(body))
+      (this.targetPositionGetter.apply(body))
          .ifPresent(positionTracker -> BehaviorUtils.setWalkAndLookTargetMemories(body, positionTracker, this.speedModifier, 3));
    }
 
