From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/behavior/InteractWithDoor.java b/net/minecraft/world/entity/ai/behavior/InteractWithDoor.java
index 502d21eb7d91144b36d5f24f3a968e3af4ab423d..3e13be34ed52787dd67feb661af4177a25fa89b1 100644
--- a/net/minecraft/world/entity/ai/behavior/InteractWithDoor.java
+++ b/net/minecraft/world/entity/ai/behavior/InteractWithDoor.java
@@ -116,7 +116,7 @@ public class InteractWithDoor {
    private static boolean areOtherMobsComingThroughDoor(final LivingEntity body, final BlockPos doorPos, final Optional<List<LivingEntity>> nearestEntities) {
       return nearestEntities.isEmpty()
          ? false
-         : ((List)nearestEntities.get())
+         : (nearestEntities.get())
             .stream()
             .filter(otherMob -> otherMob.getType() == body.getType())
             .filter(otherMob -> doorPos.closerToCenterThan(otherMob.position(), 2.0))
