From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/behavior/PoiCompetitorScan.java b/net/minecraft/world/entity/ai/behavior/PoiCompetitorScan.java
index 0a9163d21d84ec2870eff5dedecf8e691201277a..58c075a8ca79a733a84f9dfca1ccea8d58c13338 100644
--- a/net/minecraft/world/entity/ai/behavior/PoiCompetitorScan.java
+++ b/net/minecraft/world/entity/ai/behavior/PoiCompetitorScan.java
@@ -22,7 +22,7 @@ public class PoiCompetitorScan {
                   level.getPoiManager()
                      .getType(pos.pos())
                      .ifPresent(
-                        poiType -> i.<List>get(nearestEntities)
+                             poiType -> i.<List<LivingEntity>>get(nearestEntities)
                            .stream()
                            .filter(v -> v instanceof Villager && v != body)
                            .map(v -> (Villager)v)
