From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/server/jsonrpc/methods/DiscoveryService.java b/net/minecraft/server/jsonrpc/methods/DiscoveryService.java
index 412b10ef9a227d925098454b98e407665afb8ec7..827a378ded4e89021452243ce1082315b4b13f0e 100644
--- a/net/minecraft/server/jsonrpc/methods/DiscoveryService.java
+++ b/net/minecraft/server/jsonrpc/methods/DiscoveryService.java
@@ -42,7 +42,7 @@ public class DiscoveryService {
 
       private static MapCodec<DiscoveryService.DiscoverComponents> typedSchema() {
          return RecordCodecBuilder.mapCodec(
-            i -> i.group(Codec.unboundedMap(Codec.STRING, Schema.CODEC).fieldOf("schemas").forGetter(DiscoveryService.DiscoverComponents::schemas))
+                 i -> i.group(Codec.unboundedMap(Codec.STRING, (Codec<Schema<?>>) Schema.CODEC).fieldOf("schemas").forGetter(DiscoveryService.DiscoverComponents::schemas))
                .apply(i, DiscoveryService.DiscoverComponents::new)
          );
       }
