--- a/net/minecraft/server/packs/resources/ResourceMetadata.java
+++ b/net/minecraft/server/packs/resources/ResourceMetadata.java
@@ -74,7 +_,7 @@
    }
 
    default List<MetadataSectionType.WithValue<?>> getTypedSections(final Collection<MetadataSectionType<?>> types) {
-      return (List<MetadataSectionType.WithValue<?>>)types.stream()
+      return types.stream()
          .map(this::getTypedSection)
          .flatMap(Optional::stream)
          .collect(Collectors.toUnmodifiableList());
@@ -89,7 +_,7 @@
 
       @Override
       public <T> Optional<T> getSection(final MetadataSectionType<T> serializer) {
-         return Optional.ofNullable(this.values.get(serializer));
+         return Optional.ofNullable((T)this.values.get(serializer));
       }
    }
 }
