From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/core/component/DataComponentPatch.java b/net/minecraft/core/component/DataComponentPatch.java
index 1cdb1eb485825cf787058f2438c1dd32d827994b..5ef95e2d01294f6a2c59507f685be39c0cd6f7de 100644
--- a/net/minecraft/core/component/DataComponentPatch.java
+++ b/net/minecraft/core/component/DataComponentPatch.java
@@ -22,15 +22,15 @@ import org.jspecify.annotations.Nullable;
 
 public final class DataComponentPatch {
    public static final DataComponentPatch EMPTY = new DataComponentPatch(Reference2ObjectMaps.emptyMap());
-   public static final Codec<DataComponentPatch> CODEC = Codec.dispatchedMap(DataComponentPatch.PatchKey.CODEC, DataComponentPatch.PatchKey::valueCodec)
+   public static final Codec<DataComponentPatch> CODEC = Codec.<PatchKey, Object>dispatchedMap(DataComponentPatch.PatchKey.CODEC, DataComponentPatch.PatchKey::valueCodec)
       .xmap(data -> {
          if (data.isEmpty()) {
             return EMPTY;
          } else {
-            Reference2ObjectMap<DataComponentType<?>, Optional<?>> map = new Reference2ObjectArrayMap(data.size());
+            Reference2ObjectMap<DataComponentType<?>, Optional<?>> map = new Reference2ObjectArrayMap<>(data.size());
 
             for (Entry<DataComponentPatch.PatchKey, ?> entry : data.entrySet()) {
-               DataComponentPatch.PatchKey key = (DataComponentPatch.PatchKey)entry.getKey();
+               DataComponentPatch.PatchKey key = entry.getKey();
                if (key.removed()) {
                   map.put(key.type(), Optional.empty());
                } else {
@@ -41,14 +41,12 @@ public final class DataComponentPatch {
             return new DataComponentPatch(map);
          }
       }, patch -> {
-         Reference2ObjectMap<DataComponentPatch.PatchKey, Object> map = new Reference2ObjectArrayMap(patch.map.size());
-         Iterator i$ = Reference2ObjectMaps.fastIterable(patch.map).iterator();
+         Reference2ObjectMap<DataComponentPatch.PatchKey, Object> map = new Reference2ObjectArrayMap<>(patch.map.size());
 
-         while (i$.hasNext()) {
-            Entry<DataComponentType<?>, Optional<?>> entry = (Entry<DataComponentType<?>, Optional<?>>)i$.next();
-            DataComponentType<?> type = (DataComponentType<?>)entry.getKey();
+         for (Entry<DataComponentType<?>, Optional<?>> entry : Reference2ObjectMaps.fastIterable(patch.map)) {
+            DataComponentType<?> type = entry.getKey();
             if (!type.isTransient()) {
-               Optional<?> value = (Optional<?>)entry.getValue();
+               Optional<?> value = entry.getValue();
                if (value.isPresent()) {
                   map.put(new DataComponentPatch.PatchKey(type, false), value.get());
                } else {
