--- a/net/minecraft/world/entity/ai/attributes/DefaultAttributes.java
+++ b/net/minecraft/world/entity/ai/attributes/DefaultAttributes.java
@@ -91,7 +_,7 @@
 
 public class DefaultAttributes {
    private static final Logger LOGGER = LogUtils.getLogger();
-   private static final Map<EntityType<? extends LivingEntity>, AttributeSupplier> SUPPLIERS = ImmutableMap.builder()
+   private static final Map<EntityType<? extends LivingEntity>, AttributeSupplier> SUPPLIERS = ImmutableMap.<EntityType<? extends LivingEntity>, AttributeSupplier>builder()
       .put(EntityType.ALLAY, Allay.createAttributes().build())
       .put(EntityType.ARMADILLO, Armadillo.createAttributes().build())
       .put(EntityType.ARMOR_STAND, ArmorStand.createAttributes().build())
