From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/memory/MemoryMap.java b/net/minecraft/world/entity/ai/memory/MemoryMap.java
index 4df62a8ad8602626c683d801bb5c11d6fc860d5c..d4d1f918194b2c363b21a55d355d3f468a948752 100644
--- a/net/minecraft/world/entity/ai/memory/MemoryMap.java
+++ b/net/minecraft/world/entity/ai/memory/MemoryMap.java
@@ -15,7 +15,7 @@ public final class MemoryMap implements Iterable<MemoryMap.Value<?>> {
    private static final Codec<MemoryModuleType<?>> SERIALIZABLE_MEMORY_MODULE_CODEC = BuiltInRegistries.MEMORY_MODULE_TYPE
       .byNameCodec()
       .validate(type -> type.canSerialize() ? DataResult.success(type) : DataResult.error(() -> "Memory module " + type + " cannot be encoded"));
-   public static final Codec<MemoryMap> CODEC = Codec.dispatchedMap(SERIALIZABLE_MEMORY_MODULE_CODEC, type -> (Codec)type.getCodec().orElseThrow())
+   public static final Codec<MemoryMap> CODEC = Codec.<MemoryModuleType<?>, ExpirableValue<?>>dispatchedMap(SERIALIZABLE_MEMORY_MODULE_CODEC, type -> type.getCodec().orElseThrow())
       .xmap(MemoryMap::new, m -> m.memories);
    public static final MemoryMap EMPTY = new MemoryMap(Map.of());
    private final Map<MemoryModuleType<?>, ExpirableValue<?>> memories;
@@ -25,7 +25,7 @@ public final class MemoryMap implements Iterable<MemoryMap.Value<?>> {
    }
 
    public static MemoryMap of(final Stream<MemoryMap.Value<?>> memories) {
-      return new MemoryMap((Map<MemoryModuleType<?>, ExpirableValue<?>>)memories.collect(Collectors.toMap(MemoryMap.Value::type, MemoryMap.Value::value)));
+      return new MemoryMap(memories.collect(Collectors.toMap(MemoryMap.Value::type, MemoryMap.Value::value)));
    }
 
    public <U> @Nullable ExpirableValue<U> get(final MemoryModuleType<U> type) {
