From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/storage/loot/functions/CopyComponentsFunction.java b/net/minecraft/world/level/storage/loot/functions/CopyComponentsFunction.java
index c6e6340d733033bf424bc65455257b2a13b29c73..e1210e39c1354df2388d1c1edd66dc6b413aecf6 100644
--- a/net/minecraft/world/level/storage/loot/functions/CopyComponentsFunction.java
+++ b/net/minecraft/world/level/storage/loot/functions/CopyComponentsFunction.java
@@ -82,7 +82,7 @@ public class CopyComponentsFunction extends LootItemConditionalFunction {
             itemStack.applyComponents(sourceComponents.filter(this.bakedPredicate));
          } else {
             Collection<DataComponentType<?>> exclude = (Collection<DataComponentType<?>>)this.exclude.orElse(List.of());
-            ((Stream)this.include.map(Collection::stream).orElse(BuiltInRegistries.DATA_COMPONENT_TYPE.listElements().map(Holder::value)))
+            (this.include.map(Collection::stream).orElse(BuiltInRegistries.DATA_COMPONENT_TYPE.listElements().map(Holder::value)))
                .forEach(componentType -> {
                   if (!exclude.contains(componentType)) {
                      TypedDataComponent<?> value = data.getTyped(componentType);
