--- a/net/minecraft/world/level/block/state/properties/EnumProperty.java
+++ b/net/minecraft/world/level/block/state/properties/EnumProperty.java
@@ -46,7 +_,7 @@
 
    @Override
    public Optional<T> getValue(final String name) {
-      return Optional.ofNullable((Enum)this.names.get(name));
+      return Optional.ofNullable(this.names.get(name));
    }
 
    public String getName(final T value) {
@@ -77,7 +_,7 @@
    }
 
    public static <T extends Enum<T> & StringRepresentable> EnumProperty<T> create(final String name, final Class<T> clazz, final Predicate<T> filter) {
-      return create(name, clazz, (List<T>)Arrays.stream((Enum[])clazz.getEnumConstants()).filter(filter).collect(Collectors.toList()));
+      return create(name, clazz, (List<T>)Arrays.stream(clazz.getEnumConstants()).filter(filter).collect(Collectors.toList()));
    }
 
    @SafeVarargs
