From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/core/MappedRegistry.java b/net/minecraft/core/MappedRegistry.java
index 7b75c0f08e2c6bd4be3b458630f091750184b2e6..93c12c1f57781b68f927e68bc229c875f506a16e 100644
--- a/net/minecraft/core/MappedRegistry.java
+++ b/net/minecraft/core/MappedRegistry.java
@@ -89,9 +89,9 @@ public class MappedRegistry<T> implements WritableRegistry<T> {
       Objects.requireNonNull(key);
       Objects.requireNonNull(value);
       if (this.byLocation.containsKey(key.identifier())) {
-         throw (IllegalStateException)Util.pauseInIde((T)(new IllegalStateException("Adding duplicate key '" + key + "' to registry")));
+         throw Util.pauseInIde((new IllegalStateException("Adding duplicate key '" + key + "' to registry")));
       } else if (this.byValue.containsKey(value)) {
-         throw (IllegalStateException)Util.pauseInIde((T)(new IllegalStateException("Adding duplicate value '" + value + "' to registry")));
+         throw Util.pauseInIde((new IllegalStateException("Adding duplicate value '" + value + "' to registry")));
       } else {
          Holder.Reference<T> holder;
          if (this.unregisteredIntrusiveHolders != null) {
