From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/core/RegistrySetBuilder.java b/net/minecraft/core/RegistrySetBuilder.java
index e492c6b682284a9d29449e3ad487b1ab16d9c395..94a5d9f8e5a53905bb3b5a38b53a5f6b84e70a30 100644
--- a/net/minecraft/core/RegistrySetBuilder.java
+++ b/net/minecraft/core/RegistrySetBuilder.java
@@ -120,9 +120,6 @@ public class RegistrySetBuilder {
          @Override
          public <V> RegistryOps<V> createSerializationContext(final DynamicOps<V> parent) {
             return RegistryOps.create(parent, new RegistryOps.RegistryInfoLookup() {
-               {
-                  Objects.requireNonNull(<VAR_NAMELESS_ENCLOSURE>);
-               }
 
                @Override
                public <T> Optional<RegistryOps.RegistryInfo<T>> lookup(final ResourceKey<? extends Registry<? extends T>> registryKey) {
@@ -152,7 +149,7 @@ public class RegistrySetBuilder {
    ) {
       RegistrySetBuilder.UniversalOwner fullPatchedOwner = new RegistrySetBuilder.UniversalOwner();
       MutableObject<HolderLookup.Provider> resultReference = new MutableObject();
-      List<HolderLookup.RegistryLookup<?>> lazyFullRegistries = (List<HolderLookup.RegistryLookup<?>>)newRegistries.keySet()
+      List<HolderLookup.RegistryLookup<?>> lazyFullRegistries = newRegistries.keySet()
          .stream()
          .map(
             registryKey -> this.createLazyFullPatchedRegistries(
@@ -399,7 +396,7 @@ public class RegistrySetBuilder {
             if (key.isFor(this.key)) {
                RegistrySetBuilder.RegisteredValue<T> value = (RegistrySetBuilder.RegisteredValue<T>)entry.getValue();
                Holder.Reference<T> holder = (Holder.Reference<T>)state.lookup.holders.remove(key);
-               result.put(key, new RegistrySetBuilder.ValueAndHolder<>(value, Optional.ofNullable(holder)));
+               result.put((ResourceKey<T>)key, new RegistrySetBuilder.ValueAndHolder<>(value, Optional.ofNullable(holder)));
                iterator.remove();
             }
          }
@@ -421,13 +418,13 @@ public class RegistrySetBuilder {
       }
 
       private <T> Holder.Reference<T> getOrCreate(final ResourceKey<T> id) {
-         return (Holder.Reference<T>)this.holders.computeIfAbsent(id, k -> Holder.Reference.createStandAlone(this.owner, k));
+         return (Holder.Reference<T>)this.holders.computeIfAbsent((ResourceKey<Object>) id, k -> Holder.Reference.createStandAlone(this.owner, (ResourceKey<Object>)k));
       }
    }
 
    private static class UniversalOwner implements HolderOwner<Object> {
       public <T> HolderOwner<T> cast() {
-         return this;
+         return (HolderOwner<T>) this;
       }
    }
 
