From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/animal/fox/Fox.java b/net/minecraft/world/entity/animal/fox/Fox.java
index 5d19dca59c50d628fa052185053649c7cca9c33e..4faac61335e98b0c99adfa12dacc80ada04c1a85 100644
--- a/net/minecraft/world/entity/animal/fox/Fox.java
+++ b/net/minecraft/world/entity/animal/fox/Fox.java
@@ -129,7 +129,7 @@ public class Fox extends Animal {
    private static final Predicate<Entity> AVOID_PLAYERS = entity -> !entity.isDiscrete() && EntitySelector.NO_CREATIVE_OR_SPECTATOR.test(entity);
    private static final int MIN_TICKS_BEFORE_EAT = 600;
    private static final EntityDimensions BABY_DIMENSIONS = EntityType.FOX.getDimensions().scale(0.6F).withEyeHeight(0.2975F);
-   private static final Codec<List<EntityReference<LivingEntity>>> TRUSTED_LIST_CODEC = EntityReference.codec().listOf();
+   private static final Codec<List<EntityReference<LivingEntity>>> TRUSTED_LIST_CODEC = EntityReference.<LivingEntity>codec().listOf();
    private static final boolean DEFAULT_SLEEPING = false;
    private static final boolean DEFAULT_SITTING = false;
    private static final boolean DEFAULT_CROUCHING = false;
@@ -178,7 +178,7 @@ public class Fox extends Animal {
       this.goalSelector.addGoal(3, new Fox.FoxBreedGoal(1.0));
       this.goalSelector
          .addGoal(
-            4, new AvoidEntityGoal(this, Player.class, 16.0F, 1.6, 1.4, entity -> AVOID_PLAYERS.test(entity) && !this.trusts(entity) && !this.isDefending())
+            4, new AvoidEntityGoal(this, Player.class, 16.0F, 1.6, 1.4, entity -> AVOID_PLAYERS.test((Entity) entity) && !this.trusts((LivingEntity) entity) && !this.isDefending())
          );
       this.goalSelector.addGoal(4, new AvoidEntityGoal(this, Wolf.class, 8.0F, 1.6, 1.4, entity -> !((Wolf)entity).isTame() && !this.isDefending()));
       this.goalSelector.addGoal(4, new AvoidEntityGoal(this, PolarBear.class, 8.0F, 1.6, 1.4, entity -> !this.isDefending()));
@@ -438,7 +438,7 @@ public class Fox extends Animal {
    protected void readAdditionalSaveData(final ValueInput input) {
       super.readAdditionalSaveData(input);
       this.clearTrusted();
-      ((List)input.read("Trusted", TRUSTED_LIST_CODEC).orElse(List.of())).forEach(this::addTrustedEntity);
+      (input.read("Trusted", TRUSTED_LIST_CODEC).orElse(List.of())).forEach(this::addTrustedEntity);
       this.setSleeping(input.getBooleanOr("Sleeping", false));
       this.setVariant((Fox.Variant)input.read("Type", Fox.Variant.CODEC).orElse(Fox.Variant.DEFAULT));
       this.setSitting(input.getBooleanOr("Sitting", false));
