From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/animal/sniffer/SnifferAi.java b/net/minecraft/world/entity/animal/sniffer/SnifferAi.java
index 86fbf3469e37724465951a2bb9c2ee13fdaef8a3..83e22f1dab667d26383d85b7597124910cfcb25e 100644
--- a/net/minecraft/world/entity/animal/sniffer/SnifferAi.java
+++ b/net/minecraft/world/entity/animal/sniffer/SnifferAi.java
@@ -56,7 +56,7 @@ public class SnifferAi {
    }
 
    private static ActivityData<Sniffer> initCoreActivity() {
-      return ActivityData.create(Activity.CORE, 0, ImmutableList.of(new Swim(0.8F), new AnimalPanic<Sniffer>(2.0F) {
+      return ActivityData.create(Activity.CORE, 0, ImmutableList.of(new Swim<>(0.8F), new AnimalPanic<Sniffer>(2.0F) {
          protected void start(final ServerLevel level, final Sniffer body, final long timestamp) {
             SnifferAi.resetSniffing(body);
             super.start(level, body, timestamp);
