From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/core/component/predicates/JukeboxPlayablePredicate.java b/net/minecraft/core/component/predicates/JukeboxPlayablePredicate.java
index bfaac9e8777ef71bca675847754f3e2913b1a8fc..613aa1816a726e8721502bf849e6797ca4ef1cfd 100644
--- a/net/minecraft/core/component/predicates/JukeboxPlayablePredicate.java
+++ b/net/minecraft/core/component/predicates/JukeboxPlayablePredicate.java
@@ -31,7 +31,7 @@ public record JukeboxPlayablePredicate(Optional<HolderSet<JukeboxSong>> song) im
       } else {
          boolean songIsPresent = false;
 
-         for (Holder<JukeboxSong> maybeSong : (HolderSet)this.song.get()) {
+         for (Holder<JukeboxSong> maybeSong : this.song.get()) {
             Optional<ResourceKey<JukeboxSong>> songId = maybeSong.unwrapKey();
             if (!songId.isEmpty() && songId.equals(value.song().unwrapKey())) {
                songIsPresent = true;
