From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/block/state/StateDefinition.java b/net/minecraft/world/level/block/state/StateDefinition.java
index 11c2a475ff49ee28b8dc0979be76cf63ae52da06..13ea89dd2e43251bc24f2a4be3b8f0c9b985aae2 100644
--- a/net/minecraft/world/level/block/state/StateDefinition.java
+++ b/net/minecraft/world/level/block/state/StateDefinition.java
@@ -56,7 +56,7 @@ public class StateDefinition<O, S extends StateHolder<O, S>> {
    private static <O, S extends StateHolder<O, S>> MapCodec<S> createCodec(
       final O owner, final Function<O, S> defaultState, final Map<String, Property<?>> propertiesByName
    ) {
-      Supplier<S> defaultSupplier = () -> (StateHolder)defaultState.apply(owner);
+      Supplier<S> defaultSupplier = () -> defaultState.apply(owner);
       MapCodec<S> codec = MapCodec.unit(defaultSupplier);
 
       for (Entry<String, Property<?>> entry : propertiesByName.entrySet()) {
@@ -137,7 +137,7 @@ public class StateDefinition<O, S extends StateHolder<O, S>> {
    ) {
       return Codec.mapPair(codec, property.valueCodec().fieldOf(name).orElseGet(var0 -> {}, () -> property.value((StateHolder<?, ?>)defaultSupplier.get())))
          .xmap(
-            pair -> (StateHolder)((StateHolder)pair.getFirst()).setValue(property, ((Property.Value)pair.getSecond()).value()),
+            pair -> (pair.getFirst()).setValue(property, (pair.getSecond()).value()),
             state -> Pair.of(state, property.value(state))
          );
    }
