From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/valueproviders/BiasedToBottomInt.java b/net/minecraft/util/valueproviders/BiasedToBottomInt.java
index 33a9f8c127de969afc7919d01401019a221d7bbb..3aea71691ac2479e8c96261106e14b0f0ced3761 100644
--- a/net/minecraft/util/valueproviders/BiasedToBottomInt.java
+++ b/net/minecraft/util/valueproviders/BiasedToBottomInt.java
@@ -7,7 +7,7 @@ import com.mojang.serialization.codecs.RecordCodecBuilder;
 import net.minecraft.util.RandomSource;
 
 public record BiasedToBottomInt(int minInclusive, int maxInclusive) implements IntProvider {
-   public static final MapCodec<BiasedToBottomInt> MAP_CODEC = RecordCodecBuilder.mapCodec(
+   public static final MapCodec<BiasedToBottomInt> MAP_CODEC = RecordCodecBuilder.<BiasedToBottomInt>mapCodec(
          i -> i.group(
                Codec.INT.fieldOf("min_inclusive").forGetter(BiasedToBottomInt::minInclusive),
                Codec.INT.fieldOf("max_inclusive").forGetter(BiasedToBottomInt::maxInclusive)
