--- a/net/minecraft/util/valueproviders/TrapezoidFloat.java
+++ b/net/minecraft/util/valueproviders/TrapezoidFloat.java
@@ -7,7 +_,7 @@
 import net.minecraft.util.RandomSource;
 
 public record TrapezoidFloat(float min, float max, float plateau) implements FloatProvider {
-   public static final MapCodec<TrapezoidFloat> MAP_CODEC = RecordCodecBuilder.mapCodec(
+   public static final MapCodec<TrapezoidFloat> MAP_CODEC = RecordCodecBuilder.<TrapezoidFloat>mapCodec(
          i -> i.group(
                Codec.FLOAT.fieldOf("min").forGetter(TrapezoidFloat::min),
                Codec.FLOAT.fieldOf("max").forGetter(TrapezoidFloat::max),
