From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/server/dialog/input/NumberRangeInput.java b/net/minecraft/server/dialog/input/NumberRangeInput.java
index cc623f47af2811cc0af88663fabf7c241a929eb3..9bc2b98f43b67298badbdaa30cb1572ad4abfa20 100644
--- a/net/minecraft/server/dialog/input/NumberRangeInput.java
+++ b/net/minecraft/server/dialog/input/NumberRangeInput.java
@@ -32,7 +32,7 @@ public record NumberRangeInput(int width, Component label, String labelFormat, N
    }
 
    public record RangeInfo(float start, float end, Optional<Float> initial, Optional<Float> step) {
-      public static final MapCodec<NumberRangeInput.RangeInfo> MAP_CODEC = RecordCodecBuilder.mapCodec(
+      public static final MapCodec<NumberRangeInput.RangeInfo> MAP_CODEC = RecordCodecBuilder.<NumberRangeInput.RangeInfo>mapCodec(
             i -> i.group(
                   Codec.FLOAT.fieldOf("start").forGetter(NumberRangeInput.RangeInfo::start),
                   Codec.FLOAT.fieldOf("end").forGetter(NumberRangeInput.RangeInfo::end),
