From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/item/enchantment/EnchantmentEffectComponents.java b/net/minecraft/world/item/enchantment/EnchantmentEffectComponents.java
index 1f0015eb004f752066daaff2d683fe4bb1a12e1b..c5946f1f5a6991babd92937b1163ec3c58782312 100644
--- a/net/minecraft/world/item/enchantment/EnchantmentEffectComponents.java
+++ b/net/minecraft/world/item/enchantment/EnchantmentEffectComponents.java
@@ -128,7 +128,7 @@ public interface EnchantmentEffectComponents {
 
    private static <T> DataComponentType<T> register(final String id, final UnaryOperator<DataComponentType.Builder<T>> builder) {
       return Registry.register(
-         BuiltInRegistries.ENCHANTMENT_EFFECT_COMPONENT_TYPE, id, ((DataComponentType.Builder)builder.apply(DataComponentType.builder())).build()
+         BuiltInRegistries.ENCHANTMENT_EFFECT_COMPONENT_TYPE, id, (builder.apply(DataComponentType.builder())).build()
       );
    }
 }
