From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/monster/creaking/CreakingAi.java b/net/minecraft/world/entity/monster/creaking/CreakingAi.java
index e67b0f2e96386f68f599fec64e238e97edab4b33..be6839de25b671fbe7c4cc472a0fe391c878ffff 100644
--- a/net/minecraft/world/entity/monster/creaking/CreakingAi.java
+++ b/net/minecraft/world/entity/monster/creaking/CreakingAi.java
@@ -30,7 +30,7 @@ public class CreakingAi {
    static ActivityData<Creaking> initCoreActivity() {
       return ActivityData.create(Activity.CORE, 0, ImmutableList.of(new Swim<Creaking>(0.8F) {
          protected boolean checkExtraStartConditions(final ServerLevel level, final Creaking body) {
-            return body.canMove() && super.checkExtraStartConditions(level, (LivingEntity)body);
+            return body.canMove() && super.checkExtraStartConditions(level, body);
          }
       }, new LookAtTargetSink(45, 90), new MoveToTargetSink()));
    }
@@ -45,7 +45,7 @@ public class CreakingAi {
                (level, mob) -> mob.getBrain().getMemory(MemoryModuleType.NEAREST_VISIBLE_ATTACKABLE_PLAYER)
             ),
             SetEntityLookTargetSometimes.create(8.0F, UniformInt.of(30, 60)),
-            new RunOne(
+            new RunOne<>(
                ImmutableList.of(
                   Pair.of(RandomStroll.stroll(0.3F), 2), Pair.of(SetWalkTargetFromLookTarget.create(0.3F, 3), 2), Pair.of(new DoNothing(30, 60), 1)
                )
