From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/commands/functions/MacroFunction.java b/net/minecraft/commands/functions/MacroFunction.java
index 4a199f889941cb6e8337b8e32b1c665bec1ad302..02a91bcb147c4a427c792e84668325c066027bd4 100644
--- a/net/minecraft/commands/functions/MacroFunction.java
+++ b/net/minecraft/commands/functions/MacroFunction.java
@@ -87,46 +87,11 @@ public class MacroFunction<T extends ExecutionCommandSource<T>> implements Comma
       Objects.requireNonNull(tag);
 
       return switch (tag) {
-         case FloatTag(float var41) -> {
-            float var25 = var41;
-            if (true) {
-               yield DECIMAL_FORMAT.format(var25);
-            }
-
-            byte var45 = 1;
-         }
-         case DoubleTag(double var39) -> {
-            double var26 = var39;
-            if (true) {
-               yield DECIMAL_FORMAT.format(var26);
-            }
-
-            byte var44 = 2;
-         }
-         case ByteTag(byte var37) -> {
-            byte var27 = var37;
-            if (true) {
-               yield String.valueOf(var27);
-            }
-
-            byte var43 = 3;
-         }
-         case ShortTag(short var35) -> {
-            short var28 = var35;
-            if (true) {
-               yield String.valueOf(var28);
-            }
-
-            byte var42 = 4;
-         }
-         case LongTag(long var33) -> {
-            long var29 = var33;
-            if (true) {
-               yield String.valueOf(var29);
-            }
-
-            byte var2 = 5;
-         }
+         case FloatTag(float floatValue) -> DECIMAL_FORMAT.format(floatValue);
+         case DoubleTag(double doubleValue) -> DECIMAL_FORMAT.format(doubleValue);
+         case ByteTag(byte byteValue) -> String.valueOf(byteValue);
+         case ShortTag(short shortValue) -> String.valueOf(shortValue);
+         case LongTag(long longValue) -> String.valueOf(longValue);
          case StringTag(String var17) -> var17;
          default -> tag.toString();
       };
