From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/animal/allay/Allay.java b/net/minecraft/world/entity/animal/allay/Allay.java
index a8810802e893776c2e3b86d4bbc62b3ec6c4f265..d8d7ddd1a72ff29e51b528fc02773fd54eccb0cd 100644
--- a/net/minecraft/world/entity/animal/allay/Allay.java
+++ b/net/minecraft/world/entity/animal/allay/Allay.java
@@ -118,7 +118,7 @@ public class Allay extends PathfinderMob implements InventoryCarrier, VibrationS
 
    @Override
    public Brain<Allay> getBrain() {
-      return super.getBrain();
+      return (Brain<Allay>) super.getBrain();
    }
 
    public static AttributeSupplier.Builder createAttributes() {
