From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/npc/villager/Villager.java b/net/minecraft/world/entity/npc/villager/Villager.java
index 4ceb8e926229694b7e5b0e5bd775761544dfe96c..69f41216b1cbdccd31e93c9f30c69dde14040859 100644
--- a/net/minecraft/world/entity/npc/villager/Villager.java
+++ b/net/minecraft/world/entity/npc/villager/Villager.java
@@ -171,13 +171,13 @@ public class Villager extends AbstractVillager implements VillagerDataHolder, Re
    );
    public static final Map<MemoryModuleType<GlobalPos>, BiPredicate<Villager, Holder<PoiType>>> POI_MEMORIES = ImmutableMap.of(
       MemoryModuleType.HOME,
-      (BiPredicate)(villager, poiType) -> poiType.is(PoiTypes.HOME),
+      (villager, poiType) -> poiType.is(PoiTypes.HOME),
       MemoryModuleType.JOB_SITE,
-      (BiPredicate)(villager, poiType) -> villager.getVillagerData().profession().value().heldJobSite().test(poiType),
+      (villager, poiType) -> villager.getVillagerData().profession().value().heldJobSite().test(poiType),
       MemoryModuleType.POTENTIAL_JOB_SITE,
-      (BiPredicate)(villager, poiType) -> VillagerProfession.ALL_ACQUIRABLE_JOBS.test(poiType),
+      (villager, poiType) -> VillagerProfession.ALL_ACQUIRABLE_JOBS.test(poiType),
       MemoryModuleType.MEETING_POINT,
-      (BiPredicate)(villager, poiType) -> poiType.is(PoiTypes.MEETING)
+      (villager, poiType) -> poiType.is(PoiTypes.MEETING)
    );
 
    public Villager(final EntityType<? extends Villager> type, final Level level) {
@@ -199,7 +199,7 @@ public class Villager extends AbstractVillager implements VillagerDataHolder, Re
 
    @Override
    public Brain<Villager> getBrain() {
-      return super.getBrain();
+      return (Brain<Villager>) super.getBrain();
    }
 
    @Override
