From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/block/state/StateHolder.java b/net/minecraft/world/level/block/state/StateHolder.java
index 65381b5cacc4d4f472f134c20aded66c4a6f3433..0ba293d0e4c8b64d4d022b51d68112e47bb9886f 100644
--- a/net/minecraft/world/level/block/state/StateHolder.java
+++ b/net/minecraft/world/level/block/state/StateHolder.java
@@ -154,7 +154,7 @@ public abstract class StateHolder<O, S> {
             S defaultValue = (S)defaultState.apply(o);
             return definition.isSingletonState()
                ? MapCodec.unit(defaultValue)
-               : definition.propertiesCodec().codec().lenientOptionalFieldOf("Properties").xmap(oo -> (StateHolder)oo.orElse(defaultValue), Optional::of);
+               : definition.propertiesCodec().codec().lenientOptionalFieldOf("Properties").xmap(oo -> oo.orElse(defaultValue), Optional::of);
          }
       );
    }
