From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/animal/goat/GoatAi.java b/net/minecraft/world/entity/animal/goat/GoatAi.java
index c987d35fa20018a568bc73c2e422c758d127b5d8..68b7dc11fe96fe17a2c3c141a759f564c727fbca 100644
--- a/net/minecraft/world/entity/animal/goat/GoatAi.java
+++ b/net/minecraft/world/entity/animal/goat/GoatAi.java
@@ -72,8 +72,8 @@ public class GoatAi {
          Activity.CORE,
          0,
          ImmutableList.of(
-            new Swim(0.8F),
-            new AnimalPanic(2.0F),
+            new Swim<>(0.8F),
+            new AnimalPanic<>(2.0F),
             new LookAtTargetSink(45, 90),
             new MoveToTargetSink(),
             new CountDownCooldownTicks(MemoryModuleType.TEMPTATION_COOLDOWN_TICKS),
@@ -113,7 +113,7 @@ public class GoatAi {
             Pair.of(0, new LongJumpMidJump(TIME_BETWEEN_LONG_JUMPS, SoundEvents.GOAT_STEP)),
             Pair.of(
                1,
-               new LongJumpToRandomPos(
+               new LongJumpToRandomPos<>(
                   TIME_BETWEEN_LONG_JUMPS, 5, 5, 3.5714288F, goat -> goat.isScreamingGoat() ? SoundEvents.GOAT_SCREAMING_LONG_JUMP : SoundEvents.GOAT_LONG_JUMP
                )
             )
@@ -144,7 +144,7 @@ public class GoatAi {
             ),
             Pair.of(
                1,
-               new PrepareRamNearestTarget(
+               new PrepareRamNearestTarget<>(
                   mob -> mob.isScreamingGoat() ? TIME_BETWEEN_RAMS_SCREAMER.minInclusive() : TIME_BETWEEN_RAMS.minInclusive(),
                   4,
                   7,
