From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/animal/goat/Goat.java b/net/minecraft/world/entity/animal/goat/Goat.java
index 6bc885ef5f003b5c9d9837253bac41a4f42eb866..26ae094458095e87c5c407cf79ed9459109df80c 100644
--- a/net/minecraft/world/entity/animal/goat/Goat.java
+++ b/net/minecraft/world/entity/animal/goat/Goat.java
@@ -160,7 +160,7 @@ public class Goat extends Animal {
 
    @Override
    public Brain<Goat> getBrain() {
-      return super.getBrain();
+      return (Brain<Goat>) super.getBrain();
    }
 
    @Override
