From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/tags/TagNetworkSerialization.java b/net/minecraft/tags/TagNetworkSerialization.java
index cba4198e3b3aa3df24c89cb5d02e881880f19fa3..26a0e8083919b37e46f45469477766254ee4fed6 100644
--- a/net/minecraft/tags/TagNetworkSerialization.java
+++ b/net/minecraft/tags/TagNetworkSerialization.java
@@ -21,7 +21,7 @@ public class TagNetworkSerialization {
    public static Map<ResourceKey<? extends Registry<?>>, TagNetworkSerialization.NetworkPayload> serializeTagsToNetwork(
       final LayeredRegistryAccess<RegistryLayer> registries
    ) {
-      return (Map<ResourceKey<? extends Registry<?>>, TagNetworkSerialization.NetworkPayload>)RegistrySynchronization.networkSafeRegistries(registries)
+      return RegistrySynchronization.networkSafeRegistries(registries)
          .map(e -> Pair.of(e.key(), serializeToNetwork(e.value())))
          .filter(e -> !((TagNetworkSerialization.NetworkPayload)e.getSecond()).isEmpty())
          .collect(Collectors.toMap(Pair::getFirst, Pair::getSecond));
@@ -50,7 +50,7 @@ public class TagNetworkSerialization {
       Map<TagKey<T>, List<Holder<T>>> tags = new HashMap();
       payload.tags.forEach((key, ids) -> {
          TagKey<T> tagKey = TagKey.create(registryKey, key);
-         List<Holder<T>> values = (List<Holder<T>>)ids.intStream().mapToObj(registry::get).flatMap(Optional::stream).collect(Collectors.toUnmodifiableList());
+         List<Holder<T>> values = ids.intStream().mapToObj(registry::get).flatMap(Optional::stream).collect(Collectors.toUnmodifiableList());
          tags.put(tagKey, values);
       });
       return new TagLoader.LoadResult<>(registryKey, tags);
