From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/server/jsonrpc/IncomingRpcMethods.java b/net/minecraft/server/jsonrpc/IncomingRpcMethods.java
index c3dac561efb3f7a4f1df62b8ec219a2416902e7e..3491aec066e94743b101fd8460c96bf87a622bf2 100644
--- a/net/minecraft/server/jsonrpc/IncomingRpcMethods.java
+++ b/net/minecraft/server/jsonrpc/IncomingRpcMethods.java
@@ -359,7 +359,7 @@ public class IncomingRpcMethods {
          .description("Get the available game rule keys and their current values")
          .response("gamerules", Schema.TYPED_GAME_RULE_SCHEMA.asRef().asArray())
          .register(methodRegistry, "gamerules");
-      IncomingRpcMethod.method(GameRulesService::update)
+      IncomingRpcMethod.<GameRulesService.GameRuleUpdate<?>,GameRulesService.GameRuleUpdate<?>>method(GameRulesService::update)
          .description("Update game rule value")
          .param("gamerule", Schema.UNTYPED_GAME_RULE_SCHEMA.asRef())
          .response("gamerule", Schema.TYPED_GAME_RULE_SCHEMA.asRef())
