From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/item/trading/VillagerTrades.java b/net/minecraft/world/item/trading/VillagerTrades.java
index 4a76cfb2456cfe3635cfbde3cc833f13adce38db..cf8bbd10a8bc6fec99c94616e4a0a359eb478b7e 100644
--- a/net/minecraft/world/item/trading/VillagerTrades.java
+++ b/net/minecraft/world/item/trading/VillagerTrades.java
@@ -2256,7 +2256,7 @@ public class VillagerTrades {
 
    public static HolderSet<VillagerType> villagerTypeHolderSet(final HolderGetter<VillagerType> villagerVariants, final ResourceKey<VillagerType> resourceKey) {
       Optional<Holder.Reference<VillagerType>> holder = villagerVariants.get(resourceKey);
-      return (HolderSet<VillagerType>)(holder.isPresent() ? HolderSet.direct((Holder)holder.get()) : HolderSet.empty());
+      return (holder.isPresent() ? HolderSet.direct((Holder)holder.get()) : HolderSet.empty());
    }
 
    public static HolderSet<VillagerType> villagerTypeHolderSet(
