From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/data/worldgen/placement/VegetationPlacements.java b/net/minecraft/data/worldgen/placement/VegetationPlacements.java
index d9b1b180b8ddf630cfc275f1ce655606a0a19516..d1608362c7c08664772cde54e4668559f086be67 100644
--- a/net/minecraft/data/worldgen/placement/VegetationPlacements.java
+++ b/net/minecraft/data/worldgen/placement/VegetationPlacements.java
@@ -150,7 +150,7 @@ public class VegetationPlacements {
    }
 
    private static Builder<PlacementModifier> treePlacementBase(final PlacementModifier frequency) {
-      return ImmutableList.builder()
+      return ImmutableList.<PlacementModifier>builder()
          .add(frequency)
          .add(InSquarePlacement.spread())
          .add(TREE_THRESHOLD)
