From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/commands/arguments/OperationArgument.java b/net/minecraft/commands/arguments/OperationArgument.java
index cfac854ce8428d184787890035b373becb3cfb03..ee0f6b7d8fb9f37c43f29561a8e0136f994c151b 100644
--- a/net/minecraft/commands/arguments/OperationArgument.java
+++ b/net/minecraft/commands/arguments/OperationArgument.java
@@ -54,7 +54,7 @@ public class OperationArgument implements ArgumentType<OperationArgument.Operati
    }
 
    private static OperationArgument.Operation getOperation(final String op) throws CommandSyntaxException {
-      return (OperationArgument.Operation)(op.equals("><") ? (a, b) -> {
+      return (op.equals("><") ? (a, b) -> {
          int swap = a.get();
          a.set(b.get());
          b.set(swap);
