From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/commands/arguments/ResourceOrIdArgument.java b/net/minecraft/commands/arguments/ResourceOrIdArgument.java
index 110b68e2c57ae5df1215ff610c5fbd98b1479637..677478be2b4dea0a1c59b360b8394d184af23a5b 100644
--- a/net/minecraft/commands/arguments/ResourceOrIdArgument.java
+++ b/net/minecraft/commands/arguments/ResourceOrIdArgument.java
@@ -65,7 +65,7 @@ public class ResourceOrIdArgument<T> implements ArgumentType<Holder<T>> {
    }
 
    public static <T, O> Grammar<ResourceOrIdArgument.Result<T, O>> createGrammar(final ResourceKey<? extends Registry<T>> registryKey, final DynamicOps<O> ops) {
-      Grammar<O> inlineValueGrammar = SnbtGrammar.createParser((DynamicOps<T>)ops);
+      Grammar<O> inlineValueGrammar = SnbtGrammar.createParser(ops);
       Dictionary<StringReader> rules = new Dictionary<>();
       Atom<ResourceOrIdArgument.Result<T, O>> result = Atom.of("result");
       Atom<Identifier> id = Atom.of("id");
