From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/monster/hoglin/Hoglin.java b/net/minecraft/world/entity/monster/hoglin/Hoglin.java
index 2e602758916ce6296bbc66278a850e39b108d069..62fd08a8d8c077d84c7e3d87dbfc4f55738b738f 100644
--- a/net/minecraft/world/entity/monster/hoglin/Hoglin.java
+++ b/net/minecraft/world/entity/monster/hoglin/Hoglin.java
@@ -128,7 +128,7 @@ public class Hoglin extends Animal implements Enemy, HoglinBase {
 
    @Override
    public Brain<Hoglin> getBrain() {
-      return super.getBrain();
+      return (Brain<Hoglin>) super.getBrain();
    }
 
    @Override
