From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/sensing/GolemSensor.java b/net/minecraft/world/entity/ai/sensing/GolemSensor.java
index ddbeeb9074e14e80234bf5b1cb8882db3a17988f..a87a88ca36bd6697a4122072b7cb1e56bd364bff 100644
--- a/net/minecraft/world/entity/ai/sensing/GolemSensor.java
+++ b/net/minecraft/world/entity/ai/sensing/GolemSensor.java
@@ -33,7 +33,7 @@ public class GolemSensor extends Sensor<LivingEntity> {
    public static void checkForNearbyGolem(final LivingEntity body) {
       Optional<List<LivingEntity>> livingEntitiesMemory = body.getBrain().getMemory(MemoryModuleType.NEAREST_LIVING_ENTITIES);
       if (!livingEntitiesMemory.isEmpty()) {
-         boolean golemPresent = ((List)livingEntitiesMemory.get()).stream().anyMatch(entity -> entity.is(EntityType.IRON_GOLEM));
+         boolean golemPresent = (livingEntitiesMemory.get()).stream().anyMatch(entity -> entity.is(EntityType.IRON_GOLEM));
          if (golemPresent) {
             golemDetected(body);
          }
