From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/sensing/VillagerHostilesSensor.java b/net/minecraft/world/entity/ai/sensing/VillagerHostilesSensor.java
index d4cd017de19a93d5c4c5d2113dcf9c9d6e6274bd..7b392243e7ce54a20a435a0289fc7f3aa35876fc 100644
--- a/net/minecraft/world/entity/ai/sensing/VillagerHostilesSensor.java
+++ b/net/minecraft/world/entity/ai/sensing/VillagerHostilesSensor.java
@@ -7,7 +7,7 @@ import net.minecraft.world.entity.LivingEntity;
 import net.minecraft.world.entity.ai.memory.MemoryModuleType;
 
 public class VillagerHostilesSensor extends NearestVisibleLivingEntitySensor {
-   private static final ImmutableMap<EntityType<?>, Float> ACCEPTABLE_DISTANCE_FROM_HOSTILES = ImmutableMap.builder()
+   private static final ImmutableMap<EntityType<?>, Float> ACCEPTABLE_DISTANCE_FROM_HOSTILES = ImmutableMap.<EntityType<?>, Float>builder()
       .put(EntityType.DROWNED, 8.0F)
       .put(EntityType.EVOKER, 12.0F)
       .put(EntityType.HUSK, 8.0F)
