From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/BannerEntityCustomNameToOverrideComponentFix.java b/net/minecraft/util/datafix/fixes/BannerEntityCustomNameToOverrideComponentFix.java
index 904e78a548919feef4c1955331fe80431a9dc1cc..e66efb3b9e289d35643a50b8e5b4cb67819a9522 100644
--- a/net/minecraft/util/datafix/fixes/BannerEntityCustomNameToOverrideComponentFix.java
+++ b/net/minecraft/util/datafix/fixes/BannerEntityCustomNameToOverrideComponentFix.java
@@ -24,7 +24,7 @@ public class BannerEntityCustomNameToOverrideComponentFix extends DataFix {
       Type<?> blockEntityType = this.getInputSchema().getType(References.BLOCK_ENTITY);
       TaggedChoiceType<?> blockEntityIdFinder = this.getInputSchema().findChoiceType(References.BLOCK_ENTITY);
       OpticFinder<?> customNameFinder = blockEntityType.findField("CustomName");
-      OpticFinder<Pair<String, String>> textComponentFinder = DSL.typeFinder(this.getInputSchema().getType(References.TEXT_COMPONENT));
+      OpticFinder<Pair<String, String>> textComponentFinder = (OpticFinder<Pair<String, String>>) DSL.typeFinder(this.getInputSchema().getType(References.TEXT_COMPONENT));
       return this.fixTypeEverywhereTyped("Banner entity custom_name to item_name component fix", blockEntityType, input -> {
          Object blockEntityId = ((Pair)input.get(blockEntityIdFinder.finder())).getFirst();
          return blockEntityId.equals("minecraft:banner") ? this.fix(input, textComponentFinder, customNameFinder) : input;
