--- a/net/minecraft/util/datafix/fixes/BiomeFix.java
+++ b/net/minecraft/util/datafix/fixes/BiomeFix.java
@@ -4,7 +_,7 @@
 import java.util.Map;
 
 public class BiomeFix {
-   public static final Map<String, String> BIOMES = ImmutableMap.builder()
+   public static final Map<String, String> BIOMES = ImmutableMap.<String, String>builder()
       .put("minecraft:extreme_hills", "minecraft:mountains")
       .put("minecraft:swampland", "minecraft:swamp")
       .put("minecraft:hell", "minecraft:nether_wastes")
