--- a/net/minecraft/util/datafix/fixes/BlockEntityRenameFix.java
+++ b/net/minecraft/util/datafix/fixes/BlockEntityRenameFix.java
@@ -17,8 +_,8 @@
    }
 
    public TypeRewriteRule makeRule() {
-      TaggedChoiceType<String> oldType = this.getInputSchema().findChoiceType(References.BLOCK_ENTITY);
-      TaggedChoiceType<String> newType = this.getOutputSchema().findChoiceType(References.BLOCK_ENTITY);
+      TaggedChoiceType<String> oldType = (TaggedChoiceType<String>) this.getInputSchema().findChoiceType(References.BLOCK_ENTITY);
+      TaggedChoiceType<String> newType = (TaggedChoiceType<String>) this.getOutputSchema().findChoiceType(References.BLOCK_ENTITY);
       return this.fixTypeEverywhere(this.name, oldType, newType, ops -> input -> input.mapFirst(this.nameChangeLookup));
    }
 
