From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/CavesAndCliffsRenames.java b/net/minecraft/util/datafix/fixes/CavesAndCliffsRenames.java
index de3d43a8fda0e1c42cda36b8aba2eb9b889a3b34..f6ff9d6a8d4bef02985ea137c3048236a68acfc1 100644
--- a/net/minecraft/util/datafix/fixes/CavesAndCliffsRenames.java
+++ b/net/minecraft/util/datafix/fixes/CavesAndCliffsRenames.java
@@ -3,7 +3,7 @@ package net.minecraft.util.datafix.fixes;
 import com.google.common.collect.ImmutableMap;
 
 public final class CavesAndCliffsRenames {
-   public static final ImmutableMap<String, String> RENAMES = ImmutableMap.builder()
+   public static final ImmutableMap<String, String> RENAMES = ImmutableMap.<String, String>builder()
       .put("minecraft:badlands_plateau", "minecraft:badlands")
       .put("minecraft:bamboo_jungle_hills", "minecraft:bamboo_jungle")
       .put("minecraft:birch_forest_hills", "minecraft:birch_forest")
