--- a/net/minecraft/util/datafix/fixes/EntityPufferfishRenameFix.java
+++ b/net/minecraft/util/datafix/fixes/EntityPufferfishRenameFix.java
@@ -6,7 +_,7 @@
 import java.util.Objects;
 
 public class EntityPufferfishRenameFix extends SimplestEntityRenameFix {
-   public static final Map<String, String> RENAMED_IDS = ImmutableMap.builder()
+   public static final Map<String, String> RENAMED_IDS = ImmutableMap.<String, String>builder()
       .put("minecraft:puffer_fish_spawn_egg", "minecraft:pufferfish_spawn_egg")
       .build();
 
