--- a/net/minecraft/util/datafix/fixes/EntityCustomNameToComponentFix.java
+++ b/net/minecraft/util/datafix/fixes/EntityCustomNameToComponentFix.java
@@ -24,7 +_,7 @@
       Type<?> entityType = this.getInputSchema().getType(References.ENTITY);
       Type<?> newEntityType = this.getOutputSchema().getType(References.ENTITY);
       OpticFinder<String> idF = DSL.fieldFinder("id", NamespacedSchema.namespacedString());
-      OpticFinder<String> customNameF = entityType.findField("CustomName");
+      OpticFinder<String> customNameF = (OpticFinder<String>) entityType.findField("CustomName");
       Type<?> newCustomNameType = newEntityType.findFieldType("CustomName");
       return this.fixTypeEverywhereTyped(
          "EntityCustomNameToComponentFix", entityType, newEntityType, entity -> fixEntity(entity, newEntityType, idF, customNameF, newCustomNameType)
