From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/FixProjectileStoredItem.java b/net/minecraft/util/datafix/fixes/FixProjectileStoredItem.java
index 3a71922e30a5d411b0dac30a1680a35021ff099d..108e1c668ea32cd90760cde187147b0f2b46d61e 100644
--- a/net/minecraft/util/datafix/fixes/FixProjectileStoredItem.java
+++ b/net/minecraft/util/datafix/fixes/FixProjectileStoredItem.java
@@ -45,7 +45,7 @@ public class FixProjectileStoredItem extends DataFix {
       final String entityName, final FixProjectileStoredItem.SubFixer<?> fixer, final Type<?> inputEntityChoiceType, final Type<T> outputEntityChoiceType
    ) {
       OpticFinder<?> entityF = DSL.namedChoice(entityName, inputEntityChoiceType);
-      return input -> input.updateTyped(entityF, outputEntityChoiceType, typed -> fixer.fix(typed, outputEntityChoiceType));
+      return input -> input.updateTyped(entityF, outputEntityChoiceType, typed -> fixer.fix(typed, (Type) outputEntityChoiceType));
    }
 
    private static <T> Typed<T> fixArrow(final Typed<?> typed, final Type<T> outputType) {
