From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/ItemLoreFix.java b/net/minecraft/util/datafix/fixes/ItemLoreFix.java
index 3f4fa08414ef0ab0aee144af77a154c751d73bbb..306a27ced21f8385941f9c6162d457ca6add8011 100644
--- a/net/minecraft/util/datafix/fixes/ItemLoreFix.java
+++ b/net/minecraft/util/datafix/fixes/ItemLoreFix.java
@@ -16,7 +16,7 @@ public class ItemLoreFix extends DataFix {
 
    protected TypeRewriteRule makeRule() {
       Type<?> itemStackType = this.getInputSchema().getType(References.ITEM_STACK);
-      Type<Pair<String, String>> textComponentType = this.getInputSchema().getType(References.TEXT_COMPONENT);
+      Type<Pair<String, String>> textComponentType = (Type<Pair<String, String>>) this.getInputSchema().getType(References.TEXT_COMPONENT);
       OpticFinder<?> tagFinder = itemStackType.findField("tag");
       OpticFinder<?> displayFinder = tagFinder.type().findField("display");
       OpticFinder<?> loreFinder = displayFinder.type().findField("Lore");
