--- a/net/minecraft/util/datafix/fixes/ItemCustomNameToComponentFix.java
+++ b/net/minecraft/util/datafix/fixes/ItemCustomNameToComponentFix.java
@@ -16,7 +_,7 @@
 
    public TypeRewriteRule makeRule() {
       Type<?> itemStackType = this.getInputSchema().getType(References.ITEM_STACK);
-      Type<Pair<String, String>> textComponentType = this.getInputSchema().getType(References.TEXT_COMPONENT);
+      Type<Pair<String, String>> textComponentType = (Type<Pair<String, String>>)this.getInputSchema().getType(References.TEXT_COMPONENT);
       OpticFinder<?> tagFinder = itemStackType.findField("tag");
       OpticFinder<?> displayFinder = tagFinder.type().findField("display");
       OpticFinder<?> customNameFinder = displayFinder.type().findField("Name");
