--- a/net/minecraft/util/datafix/fixes/JigsawRotationFix.java
+++ b/net/minecraft/util/datafix/fixes/JigsawRotationFix.java
@@ -6,7 +_,7 @@
 import java.util.Map;
 
 public class JigsawRotationFix extends AbstractBlockPropertyFix {
-   private static final Map<String, String> RENAMES = ImmutableMap.builder()
+   private static final Map<String, String> RENAMES = ImmutableMap.<String, String>builder()
       .put("down", "down_south")
       .put("up", "up_north")
       .put("north", "north_up")
