--- a/net/minecraft/util/datafix/fixes/RecipesRenameningFix.java
+++ b/net/minecraft/util/datafix/fixes/RecipesRenameningFix.java
@@ -4,7 +_,7 @@
 import java.util.Map;
 
 public class RecipesRenameningFix {
-   public static final Map<String, String> RECIPES = ImmutableMap.builder()
+   public static final Map<String, String> RECIPES = ImmutableMap.<String, String>builder()
       .put("minecraft:acacia_bark", "minecraft:acacia_wood")
       .put("minecraft:birch_bark", "minecraft:birch_wood")
       .put("minecraft:dark_oak_bark", "minecraft:dark_oak_wood")
