From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/ScoreboardDisplayNameFix.java b/net/minecraft/util/datafix/fixes/ScoreboardDisplayNameFix.java
index bd7aa18c10ac10d420228e96619ea8ee03832d02..99f4c695e5c5642456ccce61be13d01f54288c84 100644
--- a/net/minecraft/util/datafix/fixes/ScoreboardDisplayNameFix.java
+++ b/net/minecraft/util/datafix/fixes/ScoreboardDisplayNameFix.java
@@ -23,7 +23,7 @@ public class ScoreboardDisplayNameFix extends DataFix {
    protected TypeRewriteRule makeRule() {
       Type<?> inputType = this.getInputSchema().getType(this.type);
       OpticFinder<?> displayNameF = inputType.findField("DisplayName");
-      OpticFinder<Pair<String, String>> textComponentF = DSL.typeFinder(this.getInputSchema().getType(References.TEXT_COMPONENT));
+      OpticFinder<Pair<String, String>> textComponentF = DSL.typeFinder((Type<Pair<String, String>>)this.getInputSchema().getType(References.TEXT_COMPONENT));
       return this.fixTypeEverywhereTyped(
          this.name,
          inputType,
