From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/TextComponentHoverAndClickEventFix.java b/net/minecraft/util/datafix/fixes/TextComponentHoverAndClickEventFix.java
index 07b1932ade67398db0d02e49ce66e681a3d63b5f..b2de53f7aceb0e643fa1b1e47a8fb211e7fae96e 100644
--- a/net/minecraft/util/datafix/fixes/TextComponentHoverAndClickEventFix.java
+++ b/net/minecraft/util/datafix/fixes/TextComponentHoverAndClickEventFix.java
@@ -24,7 +24,7 @@ public class TextComponentHoverAndClickEventFix extends DataFix {
    }
 
    protected TypeRewriteRule makeRule() {
-      Type<? extends Pair<String, ?>> hoverEventType = this.getInputSchema().getType(References.TEXT_COMPONENT).findFieldType("hoverEvent");
+      Type<? extends Pair<String, ?>> hoverEventType = (Type<? extends Pair<String, ?>>)this.getInputSchema().getType(References.TEXT_COMPONENT).findFieldType("hoverEvent");
       return this.createFixer(
          this.getInputSchema().getTypeRaw(References.TEXT_COMPONENT), this.getOutputSchema().getType(References.TEXT_COMPONENT), hoverEventType
       );
@@ -56,20 +56,20 @@ public class TextComponentHoverAndClickEventFix extends DataFix {
             oldTextComponentType,
             newTextComponentType,
             ops -> textComponent -> {
-               boolean hasHoverOrClick = (Boolean)((Either)textComponent.getSecond()).map(simple -> false, full -> {
-                  Pair<Either<H, Unit>, Dynamic<?>> hoverAndRemainder = (Pair<Either<H, Unit>, Dynamic<?>>)((Pair)full.getSecond()).getSecond();
+               boolean hasHoverOrClick = (textComponent.getSecond()).map(simple -> false, full -> {
+                  Pair<Either<H, Unit>, Dynamic<?>> hoverAndRemainder = (Pair<Either<H, Unit>, Dynamic<?>>)(full.getSecond()).getSecond();
                   boolean hasHover = ((Either)hoverAndRemainder.getFirst()).left().isPresent();
                   boolean hasClick = ((Dynamic)hoverAndRemainder.getSecond()).get("clickEvent").result().isPresent();
                   return hasHover || hasClick;
                });
-               return !hasHoverOrClick
+               return (C2) (!hasHoverOrClick
                   ? textComponent
                   : Util.writeAndReadTypedOrThrow(
                         ExtraDataFixUtils.cast(patchedInputType, textComponent, ops),
                         newTextComponentType,
                         TextComponentHoverAndClickEventFix::fixTextComponent
                      )
-                     .getValue();
+                     .getValue());
             }
          );
       }
