--- a/net/minecraft/util/datafix/fixes/VillagerDataFix.java
+++ b/net/minecraft/util/datafix/fixes/VillagerDataFix.java
@@ -14,7 +_,7 @@
 
    @Override
    protected Typed<?> fix(final Typed<?> entity) {
-      Dynamic<?> remainder = (Dynamic<?>)entity.get(DSL.remainderFinder());
+      Dynamic<?> remainder = entity.get(DSL.remainderFinder());
       return entity.set(
          DSL.remainderFinder(),
          remainder.remove("Profession")
@@ -29,7 +_,7 @@
                      remainder.createString("profession"),
                      remainder.createString(upgradeData(remainder.get("Profession").asInt(0), remainder.get("Career").asInt(0))),
                      remainder.createString("level"),
-                     (Dynamic)DataFixUtils.orElse(remainder.get("CareerLevel").result(), remainder.createInt(1))
+                     DataFixUtils.orElse(remainder.get("CareerLevel").result(), remainder.createInt(1))
                   )
                )
             )
