From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/gametest/framework/TestFinder.java b/net/minecraft/gametest/framework/TestFinder.java
index 1b99812118fd41e996fbb9ccec32d089db30b0ed..b048ab05dbe2684934ee7fe26f940a6278682826 100644
--- a/net/minecraft/gametest/framework/TestFinder.java
+++ b/net/minecraft/gametest/framework/TestFinder.java
@@ -79,8 +79,8 @@ public class TestFinder implements TestInstanceFinder, TestPosFinder {
       private TestFinder build(final CommandSourceStack source, final TestInstanceFinder testInstanceFinder, final TestPosFinder testPosFinder) {
          return new TestFinder(
             source,
-            ((Supplier)this.testFinderWrapper.apply(testInstanceFinder::findTests))::get,
-            ((Supplier)this.structureBlockPosFinderWrapper.apply(testPosFinder::findTestPos))::get
+            (this.testFinderWrapper.apply(testInstanceFinder::findTests))::get,
+            (this.structureBlockPosFinderWrapper.apply(testPosFinder::findTestPos))::get
          );
       }
 
