From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/gametest/framework/TestEnvironmentDefinition.java b/net/minecraft/gametest/framework/TestEnvironmentDefinition.java
index 3b72936803c5cdd6901b32e0f5b0c4acf8f3d447..afe056a8b33e1e97ccda7e98e19bb5c526aa42bd 100644
--- a/net/minecraft/gametest/framework/TestEnvironmentDefinition.java
+++ b/net/minecraft/gametest/framework/TestEnvironmentDefinition.java
@@ -87,7 +87,7 @@ public interface TestEnvironmentDefinition<SavedDataType> {
       }
 
       public List<? extends TestEnvironmentDefinition.Activation<?>> setup(final ServerLevel level) {
-         return this.definitions.stream().map(b -> TestEnvironmentDefinition.activate((TestEnvironmentDefinition)b.value(), level)).toList();
+         return this.definitions.stream().map(b -> TestEnvironmentDefinition.activate(b.value(), level)).toList();
       }
 
       public void teardown(final ServerLevel level, final List<? extends TestEnvironmentDefinition.Activation<?>> activations) {
