From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/scores/Scoreboard.java b/net/minecraft/world/scores/Scoreboard.java
index 22d4db2060c8290c4068ab6186c5f0c4501cf7b7..0809a7147d730f9bfe091fe96ed954fe744a879d 100644
--- a/net/minecraft/world/scores/Scoreboard.java
+++ b/net/minecraft/world/scores/Scoreboard.java
@@ -61,7 +61,7 @@ public class Scoreboard {
    }
 
    public final void forAllObjectives(final ObjectiveCriteria criteria, final ScoreHolder name, final Consumer<ScoreAccess> operation) {
-      ((List)this.objectivesByCriteria.getOrDefault(criteria, Collections.emptyList()))
+      (this.objectivesByCriteria.getOrDefault(criteria, Collections.emptyList()))
          .forEach(o -> operation.accept(this.getOrCreatePlayerScore(name, o, true)));
    }
 
