From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/valueproviders/ClampedNormalFloat.java b/net/minecraft/util/valueproviders/ClampedNormalFloat.java
index 73ca5a3ed5943e88cdfb417b14e720e21ddf2c8a..67c6117fb123eb87f54131c45f33e766486a7c0f 100644
--- a/net/minecraft/util/valueproviders/ClampedNormalFloat.java
+++ b/net/minecraft/util/valueproviders/ClampedNormalFloat.java
@@ -8,7 +8,7 @@ import net.minecraft.util.Mth;
 import net.minecraft.util.RandomSource;
 
 public record ClampedNormalFloat(float mean, float deviation, float min, float max) implements FloatProvider {
-   public static final MapCodec<ClampedNormalFloat> MAP_CODEC = RecordCodecBuilder.mapCodec(
+   public static final MapCodec<ClampedNormalFloat> MAP_CODEC = RecordCodecBuilder.<ClampedNormalFloat>mapCodec(
          i -> i.group(
                Codec.FLOAT.fieldOf("mean").forGetter(ClampedNormalFloat::mean),
                Codec.FLOAT.fieldOf("deviation").forGetter(ClampedNormalFloat::deviation),
