From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/valueproviders/TrapezoidInt.java b/net/minecraft/util/valueproviders/TrapezoidInt.java
index addca0f49e562877ade31b31d74d2586a2c98476..e974fca0d8e30d6a3af227797ff5b373aee4ce6e 100644
--- a/net/minecraft/util/valueproviders/TrapezoidInt.java
+++ b/net/minecraft/util/valueproviders/TrapezoidInt.java
@@ -8,7 +8,7 @@ import net.minecraft.util.Mth;
 import net.minecraft.util.RandomSource;
 
 public record TrapezoidInt(int minInclusive, int maxInclusive, int plateau) implements IntProvider {
-   public static final MapCodec<TrapezoidInt> MAP_CODEC = RecordCodecBuilder.mapCodec(
+   public static final MapCodec<TrapezoidInt> MAP_CODEC = RecordCodecBuilder.<TrapezoidInt>mapCodec(
          i -> i.group(
                Codec.INT.fieldOf("min").forGetter(TrapezoidInt::minInclusive),
                Codec.INT.fieldOf("max").forGetter(TrapezoidInt::maxInclusive),
