From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/village/poi/PoiSection.java b/net/minecraft/world/entity/ai/village/poi/PoiSection.java
index 4ab36ca05656dbf530dfb3d38d948e21938c5df9..da0c198bb526c3f2ee0265c12769ac54030bd8fc 100644
--- a/net/minecraft/world/entity/ai/village/poi/PoiSection.java
+++ b/net/minecraft/world/entity/ai/village/poi/PoiSection.java
@@ -132,7 +132,7 @@ public class PoiSection {
       if (!this.isValid) {
          Short2ObjectMap<PoiRecord> oldRecords = new Short2ObjectOpenHashMap(this.records);
          this.clear();
-         updater.accept((BiConsumer)(blockPos, poiType) -> {
+         updater.accept((blockPos, poiType) -> {
             short key = SectionPos.sectionRelativePos(blockPos);
             PoiRecord newRecord = (PoiRecord)oldRecords.computeIfAbsent(key, k -> new PoiRecord(blockPos, poiType, this.setDirty));
             this.add(newRecord);
