From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/server/jsonrpc/IncomingRpcMethod.java b/net/minecraft/server/jsonrpc/IncomingRpcMethod.java
index 6ddad66132dc46f3daa0d55a04593b6a563ffadf..9cbc869b50259a3f10b62e91c53f1572f9dbc562 100644
--- a/net/minecraft/server/jsonrpc/IncomingRpcMethod.java
+++ b/net/minecraft/server/jsonrpc/IncomingRpcMethod.java
@@ -143,7 +143,7 @@ public interface IncomingRpcMethod<Params, Result> {
                   paramsJsonElement = jsonArray.get(0);
                }
 
-               Params params = (Params)((ParamInfo)this.info.params().get())
+               Params params = (this.info.params().get())
                   .schema()
                   .codec()
                   .parse(JsonOps.INSTANCE, paramsJsonElement)
@@ -152,7 +152,7 @@ public interface IncomingRpcMethod<Params, Result> {
                if (this.info.result().isEmpty()) {
                   throw new IllegalStateException("No result codec defined");
                } else {
-                  return (JsonElement)((ResultInfo)this.info.result().get())
+                  return (this.info.result().get())
                      .schema()
                      .codec()
                      .encodeStart(JsonOps.INSTANCE, result)
