From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/nbt/SnbtGrammar.java b/net/minecraft/nbt/SnbtGrammar.java
index 2f85493f94762349d77dc3464a1a7f80b1aa401c..f0ec8d20f891d446e1119448f61fcf370fe633e2 100644
--- a/net/minecraft/nbt/SnbtGrammar.java
+++ b/net/minecraft/nbt/SnbtGrammar.java
@@ -775,7 +775,7 @@ public class SnbtGrammar {
             state.errorCollector().store(state.mark(), SnbtGrammar.ERROR_INVALID_ARRAY_ELEMENT_TYPE);
             return null;
          } else {
-            return entry.create(JavaOps.INSTANCE, actualType, state);
+            return (Number) entry.create(JavaOps.INSTANCE, actualType, state);
          }
       }
 
