--- a/net/minecraft/nbt/SnbtGrammar.java
+++ b/net/minecraft/nbt/SnbtGrammar.java
@@ -775,7 +_,7 @@
             state.errorCollector().store(state.mark(), SnbtGrammar.ERROR_INVALID_ARRAY_ELEMENT_TYPE);
             return null;
          } else {
-            return entry.create(JavaOps.INSTANCE, actualType, state);
+            return (Number) entry.create(JavaOps.INSTANCE, actualType, state);
          }
       }
 
