From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java b/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java
index c92417dd88639dff3a22bfa315e87c5089149f29..3ce4c86f8fa3f4a27d722b928f727ca558e306ca 100644
--- a/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java
+++ b/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java
@@ -54,7 +54,7 @@ public class VibrationSelector {
       if (this.currentVibrationData.isEmpty()) {
          return Optional.empty();
       } else {
-         return ((Pair)this.currentVibrationData.get()).getRight() < time
+         return (this.currentVibrationData.get()).getRight() < time
             ? Optional.of((VibrationInfo)((Pair)this.currentVibrationData.get()).getLeft())
             : Optional.empty();
       }
