From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java b/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java
index c92417dd88639dff3a22bfa315e87c5089149f29..3ce4c86f8fa3f4a27d722b928f727ca558e306ca 100644
--- a/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java
+++ b/net/minecraft/world/level/gameevent/vibrations/VibrationSelector.java
@@ -54,7 +54,7 @@ public class VibrationSelector {
       if (this.currentVibrationData.isEmpty()) {
          return Optional.empty();
       } else {
-         return ((Pair)this.currentVibrationData.get()).getRight() < time
+         return (this.currentVibrationData.get()).getRight() < time
             ? Optional.of((VibrationInfo)((Pair)this.currentVibrationData.get()).getLeft())
             : Optional.empty();
       }
