From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/variant/VariantUtils.java b/net/minecraft/world/entity/variant/VariantUtils.java
index 66951dfb4786d6d0c895d3a1203ef73242b70198..30b2d0197d2a15322fbc5e5689a6b84ea14e8640 100644
--- a/net/minecraft/world/entity/variant/VariantUtils.java
+++ b/net/minecraft/world/entity/variant/VariantUtils.java
@@ -29,7 +29,7 @@ public class VariantUtils {
    }
 
    public static <T> Optional<Holder<T>> readVariant(final ValueInput input, final ResourceKey<? extends Registry<T>> registryId) {
-      return input.read("variant", (Codec<T>)Identifier.CODEC).map(id -> ResourceKey.create(registryId, id)).flatMap(input.lookup()::get);
+      return input.read("variant", Identifier.CODEC).map(id -> ResourceKey.create(registryId, id)).flatMap(input.lookup()::get);
    }
 
    public static <T extends PriorityProvider<SpawnContext, ?>> Optional<Holder.Reference<T>> selectVariantToSpawn(
