From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/StaticCache2D.java b/net/minecraft/util/StaticCache2D.java
index 3ba771566b30efdb175849f88de271fd22c2b619..ef67a3950cc86a874ad880c72ec317595d9f86a2 100644
--- a/net/minecraft/util/StaticCache2D.java
+++ b/net/minecraft/util/StaticCache2D.java
@@ -33,7 +33,7 @@ public class StaticCache2D<T> {
 
    public void forEach(final Consumer<T> consumer) {
       for (Object o : this.cache) {
-         consumer.accept(o);
+         consumer.accept((T) o);
       }
    }
 
