From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/BiomeFix.java b/net/minecraft/util/datafix/fixes/BiomeFix.java
index 0bce4c9b5b13586d558dfebdddda7387744f2326..14e90af189bb28e47ed2fec90d1f186e7e9b1436 100644
--- a/net/minecraft/util/datafix/fixes/BiomeFix.java
+++ b/net/minecraft/util/datafix/fixes/BiomeFix.java
@@ -4,7 +4,7 @@ import com.google.common.collect.ImmutableMap;
 import java.util.Map;
 
 public class BiomeFix {
-   public static final Map<String, String> BIOMES = ImmutableMap.builder()
+   public static final Map<String, String> BIOMES = ImmutableMap.<String, String>builder()
       .put("minecraft:extreme_hills", "minecraft:mountains")
       .put("minecraft:swampland", "minecraft:swamp")
       .put("minecraft:hell", "minecraft:nether_wastes")
