From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/BlockEntityRenameFix.java b/net/minecraft/util/datafix/fixes/BlockEntityRenameFix.java
index 13673e89f3d68af6ad54994fbdd657ac0b325621..05e80c90b5ccb07a5f8e1a42fc86b74de353211b 100644
--- a/net/minecraft/util/datafix/fixes/BlockEntityRenameFix.java
+++ b/net/minecraft/util/datafix/fixes/BlockEntityRenameFix.java
@@ -17,8 +17,8 @@ public class BlockEntityRenameFix extends DataFix {
    }
 
    public TypeRewriteRule makeRule() {
-      TaggedChoiceType<String> oldType = this.getInputSchema().findChoiceType(References.BLOCK_ENTITY);
-      TaggedChoiceType<String> newType = this.getOutputSchema().findChoiceType(References.BLOCK_ENTITY);
+      TaggedChoiceType<String> oldType = (TaggedChoiceType<String>) this.getInputSchema().findChoiceType(References.BLOCK_ENTITY);
+      TaggedChoiceType<String> newType = (TaggedChoiceType<String>) this.getOutputSchema().findChoiceType(References.BLOCK_ENTITY);
       return this.fixTypeEverywhere(this.name, oldType, newType, ops -> input -> input.mapFirst(this.nameChangeLookup));
    }
 
