From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/BlockEntitySignDoubleSidedEditableTextFix.java b/net/minecraft/util/datafix/fixes/BlockEntitySignDoubleSidedEditableTextFix.java
index f0ba8313dec3496b5a664886c7fef1b69447dff3..9db89dbe78e1b90a38bd3d3ca4a345ff6604381d 100644
--- a/net/minecraft/util/datafix/fixes/BlockEntitySignDoubleSidedEditableTextFix.java
+++ b/net/minecraft/util/datafix/fixes/BlockEntitySignDoubleSidedEditableTextFix.java
@@ -35,16 +35,16 @@ public class BlockEntitySignDoubleSidedEditableTextFix extends NamedEntityWriteR
 
    private static <T> Dynamic<T> fixFrontTextTag(final Dynamic<T> tag) {
       Dynamic<T> emptyLine = LegacyComponentDataFixUtils.createEmptyComponent(tag.getOps());
-      List<Dynamic<T>> lines = getLines(tag, "Text").map(line -> (Dynamic)line.orElse(emptyLine)).toList();
+      List<Dynamic<T>> lines = getLines(tag, "Text").map(line -> line.orElse(emptyLine)).toList();
       Dynamic<T> text = tag.emptyMap()
          .set("messages", tag.createList(lines.stream()))
-         .set("color", (Dynamic)tag.get("Color").result().orElse(tag.createString("black")))
-         .set("has_glowing_text", (Dynamic)tag.get("GlowingText").result().orElse(tag.createBoolean(false)));
+         .set("color", tag.get("Color").result().orElse(tag.createString("black")))
+         .set("has_glowing_text", tag.get("GlowingText").result().orElse(tag.createBoolean(false)));
       List<Optional<Dynamic<T>>> filteredLines = getLines(tag, "FilteredText").toList();
       if (filteredLines.stream().anyMatch(Optional::isPresent)) {
          text = text.set("filtered_messages", tag.createList(Streams.mapWithIndex(filteredLines.stream(), (line, index) -> {
-            Dynamic<T> fallbackLine = (Dynamic<T>)lines.get((int)index);
-            return (Dynamic)line.orElse(fallbackLine);
+            Dynamic<T> fallbackLine = lines.get((int)index);
+            return line.orElse(fallbackLine);
          })));
       }
 
