From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/ChunkStructuresTemplateRenameFix.java b/net/minecraft/util/datafix/fixes/ChunkStructuresTemplateRenameFix.java
index 11863785e18d040aa9b426ed63fa0a9b3a172432..de5edda9b85ad4d15be1c2397a31c746d617a108 100644
--- a/net/minecraft/util/datafix/fixes/ChunkStructuresTemplateRenameFix.java
+++ b/net/minecraft/util/datafix/fixes/ChunkStructuresTemplateRenameFix.java
@@ -10,19 +10,19 @@ import com.mojang.datafixers.util.Pair;
 import com.mojang.serialization.Dynamic;
 
 public class ChunkStructuresTemplateRenameFix extends DataFix {
-   private static final ImmutableMap<String, Pair<String, ImmutableMap<String, String>>> RENAMES = ImmutableMap.builder()
+   private static final ImmutableMap<String, Pair<String, ImmutableMap<String, String>>> RENAMES = ImmutableMap.<String, Pair<String, ImmutableMap<String, String>>>builder()
       .put(
          "EndCity",
          Pair.of(
             "ECP",
-            ImmutableMap.builder().put("second_floor", "second_floor_1").put("third_floor", "third_floor_1").put("third_floor_c", "third_floor_2").build()
+            ImmutableMap.<String, String>builder().put("second_floor", "second_floor_1").put("third_floor", "third_floor_1").put("third_floor_c", "third_floor_2").build()
          )
       )
       .put(
          "Mansion",
          Pair.of(
             "WMP",
-            ImmutableMap.builder()
+            ImmutableMap.<String, String>builder()
                .put("carpet_south", "carpet_south_1")
                .put("carpet_west", "carpet_west_1")
                .put("indoors_door", "indoors_door_1")
@@ -34,7 +34,7 @@ public class ChunkStructuresTemplateRenameFix extends DataFix {
          "Igloo",
          Pair.of(
             "Iglu",
-            ImmutableMap.builder()
+            ImmutableMap.<String, String>builder()
                .put("minecraft:igloo/igloo_bottom", "minecraft:igloo/bottom")
                .put("minecraft:igloo/igloo_middle", "minecraft:igloo/middle")
                .put("minecraft:igloo/igloo_top", "minecraft:igloo/top")
@@ -45,7 +45,7 @@ public class ChunkStructuresTemplateRenameFix extends DataFix {
          "Ocean_Ruin",
          Pair.of(
             "ORP",
-            ImmutableMap.builder()
+            ImmutableMap.<String, String>builder()
                .put("minecraft:ruin/big_ruin1_brick", "minecraft:underwater_ruin/big_brick_1")
                .put("minecraft:ruin/big_ruin2_brick", "minecraft:underwater_ruin/big_brick_2")
                .put("minecraft:ruin/big_ruin3_brick", "minecraft:underwater_ruin/big_brick_3")
