From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/DyeItemRenameFix.java b/net/minecraft/util/datafix/fixes/DyeItemRenameFix.java
index 16dc60a41d39f13736a5b9089991631fca0bb417..b6f2a22ea677b662fde6e3ef1b8f8be37d0a163b 100644
--- a/net/minecraft/util/datafix/fixes/DyeItemRenameFix.java
+++ b/net/minecraft/util/datafix/fixes/DyeItemRenameFix.java
@@ -4,7 +4,7 @@ import com.google.common.collect.ImmutableMap;
 import java.util.Map;
 
 public class DyeItemRenameFix {
-   public static final Map<String, String> RENAMED_IDS = ImmutableMap.builder()
+   public static final Map<String, String> RENAMED_IDS = ImmutableMap.<String, String>builder()
       .put("minecraft:cactus_green", "minecraft:green_dye")
       .put("minecraft:rose_red", "minecraft:red_dye")
       .put("minecraft:dandelion_yellow", "minecraft:yellow_dye")
