From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/EmptyItemInHotbarFix.java b/net/minecraft/util/datafix/fixes/EmptyItemInHotbarFix.java
index 890bd9173dc984eb5c74bfe4afb0c34f29a8521a..58f644353b17118d824345517cf5843c7286a11f 100644
--- a/net/minecraft/util/datafix/fixes/EmptyItemInHotbarFix.java
+++ b/net/minecraft/util/datafix/fixes/EmptyItemInHotbarFix.java
@@ -5,6 +5,7 @@ import com.mojang.datafixers.DataFix;
 import com.mojang.datafixers.OpticFinder;
 import com.mojang.datafixers.TypeRewriteRule;
 import com.mojang.datafixers.schemas.Schema;
+import com.mojang.datafixers.types.Type;
 import com.mojang.datafixers.util.Either;
 import com.mojang.datafixers.util.Pair;
 import com.mojang.datafixers.util.Unit;
@@ -16,20 +17,18 @@ public class EmptyItemInHotbarFix extends DataFix {
       super(outputSchema, false);
    }
 
-   public TypeRewriteRule makeRule() {
-      OpticFinder<Pair<String, Pair<Either<Pair<String, String>, Unit>, Pair<Either<?, Unit>, Dynamic<?>>>>> itemStackF = DSL.typeFinder(
-         this.getInputSchema().getType(References.ITEM_STACK)
-      );
-      return this.fixTypeEverywhereTyped(
-         "EmptyItemInHotbarFix",
-         this.getInputSchema().getType(References.HOTBAR),
-         input -> input.update(itemStackF, namedStack -> namedStack.mapSecond(itemStack -> {
-            Optional<String> id = ((Either)itemStack.getFirst()).left().map(Pair::getSecond);
-            Dynamic<?> remainder = (Dynamic<?>)((Pair)itemStack.getSecond()).getSecond();
-            boolean isAir = id.isEmpty() || ((String)id.get()).equals("minecraft:air");
-            boolean isEmpty = remainder.get("Count").asInt(0) <= 0;
-            return !isAir && !isEmpty ? itemStack : Pair.of(Either.right(Unit.INSTANCE), Pair.of(Either.right(Unit.INSTANCE), remainder.emptyMap()));
-         }))
-      );
-   }
+    public TypeRewriteRule makeRule() {
+        OpticFinder<Pair<String, Pair<Either<Pair<String, String>, Unit>, Pair<Either<?, Unit>, Dynamic<?>>>>> itemStackF = DSL.typeFinder((Type<Pair<String, Pair<Either<Pair<String, String>, Unit>, Pair<Either<?, Unit>, Dynamic<?>>>>>)this.getInputSchema().getType(References.ITEM_STACK));
+        return this.fixTypeEverywhereTyped(
+                "EmptyItemInHotbarFix",
+                this.getInputSchema().getType(References.HOTBAR),
+                input -> input.update(itemStackF, namedStack -> namedStack.mapSecond(itemStack -> {
+                    Optional<String> id = itemStack.getFirst().left().map(Pair::getSecond);
+                    Dynamic<?> remainder = itemStack.getSecond().getSecond();
+                    boolean isAir = id.isEmpty() || id.get().equals("minecraft:air");
+                    boolean isEmpty = remainder.get("Count").asInt(0) <= 0;
+                    return !isAir && !isEmpty ? itemStack : Pair.of(Either.right(Unit.INSTANCE), Pair.of(Either.right(Unit.INSTANCE), remainder.emptyMap()));
+                }))
+        );
+    }
 }
