From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/EntityRavagerRenameFix.java b/net/minecraft/util/datafix/fixes/EntityRavagerRenameFix.java
index dfc285688c29ed7fe3f73b205761ee25334be4a0..9ad909cacb222739786e0feed8e2bf2fe79e0e18 100644
--- a/net/minecraft/util/datafix/fixes/EntityRavagerRenameFix.java
+++ b/net/minecraft/util/datafix/fixes/EntityRavagerRenameFix.java
@@ -6,7 +6,7 @@ import java.util.Map;
 import java.util.Objects;
 
 public class EntityRavagerRenameFix extends SimplestEntityRenameFix {
-   public static final Map<String, String> RENAMED_IDS = ImmutableMap.builder().put("minecraft:illager_beast_spawn_egg", "minecraft:ravager_spawn_egg").build();
+   public static final Map<String, String> RENAMED_IDS = ImmutableMap.<String, String>builder().put("minecraft:illager_beast_spawn_egg", "minecraft:ravager_spawn_egg").build();
 
    public EntityRavagerRenameFix(final Schema outputSchema, final boolean changesType) {
       super("EntityRavagerRenameFix", outputSchema, changesType);
