From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/EntityTheRenameningFix.java b/net/minecraft/util/datafix/fixes/EntityTheRenameningFix.java
index 0119bfca8951f6d59798aa77c76a5ebca64ce0ef..2a2c101cfa186a7539b528a57a3cef8681bbd7e1 100644
--- a/net/minecraft/util/datafix/fixes/EntityTheRenameningFix.java
+++ b/net/minecraft/util/datafix/fixes/EntityTheRenameningFix.java
@@ -5,7 +5,7 @@ import com.mojang.datafixers.schemas.Schema;
 import java.util.Map;
 
 public class EntityTheRenameningFix extends SimplestEntityRenameFix {
-   public static final Map<String, String> RENAMED_IDS = ImmutableMap.builder()
+   public static final Map<String, String> RENAMED_IDS = ImmutableMap.<String, String>builder()
       .put("minecraft:commandblock_minecart", "minecraft:command_block_minecart")
       .put("minecraft:ender_crystal", "minecraft:end_crystal")
       .put("minecraft:snowman", "minecraft:snow_golem")
@@ -19,7 +19,7 @@ public class EntityTheRenameningFix extends SimplestEntityRenameFix {
       .put("minecraft:eye_of_ender_signal", "minecraft:eye_of_ender")
       .put("minecraft:fireworks_rocket", "minecraft:firework_rocket")
       .build();
-   public static final Map<String, String> RENAMED_BLOCKS = ImmutableMap.builder()
+   public static final Map<String, String> RENAMED_BLOCKS = ImmutableMap.<String, String>builder()
       .put("minecraft:portal", "minecraft:nether_portal")
       .put("minecraft:oak_bark", "minecraft:oak_wood")
       .put("minecraft:spruce_bark", "minecraft:spruce_wood")
@@ -35,7 +35,7 @@ public class EntityTheRenameningFix extends SimplestEntityRenameFix {
       .put("minecraft:stripped_dark_oak_bark", "minecraft:stripped_dark_oak_wood")
       .put("minecraft:mob_spawner", "minecraft:spawner")
       .build();
-   public static final Map<String, String> RENAMED_ITEMS = ImmutableMap.builder()
+   public static final Map<String, String> RENAMED_ITEMS = ImmutableMap.<String, String>builder()
       .putAll(RENAMED_BLOCKS)
       .put("minecraft:clownfish", "minecraft:tropical_fish")
       .put("minecraft:chorus_fruit_popped", "minecraft:popped_chorus_fruit")
