From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/EntitySpawnerItemVariantComponentFix.java b/net/minecraft/util/datafix/fixes/EntitySpawnerItemVariantComponentFix.java
index 9db6850a1df049c1e996cfcf6ec6536ffc5c1f1a..3e5da8cdc43528b5c94a4f75ca8b5ffeaee2eda5 100644
--- a/net/minecraft/util/datafix/fixes/EntitySpawnerItemVariantComponentFix.java
+++ b/net/minecraft/util/datafix/fixes/EntitySpawnerItemVariantComponentFix.java
@@ -31,9 +31,9 @@ public class EntitySpawnerItemVariantComponentFix extends DataFix {
             String id = (String)input.getOptional(idFinder).map(Pair::getSecond).orElse("");
 
             return switch (id) {
-               case "minecraft:salmon_bucket" -> input.updateTyped(componentsFinder, EntitySpawnerItemVariantComponentFix::fixSalmonBucket);
-               case "minecraft:axolotl_bucket" -> input.updateTyped(componentsFinder, EntitySpawnerItemVariantComponentFix::fixAxolotlBucket);
-               case "minecraft:tropical_fish_bucket" -> input.updateTyped(componentsFinder, EntitySpawnerItemVariantComponentFix::fixTropicalFishBucket);
+               case "minecraft:salmon_bucket" -> input.updateTyped(componentsFinder, (Fixer) EntitySpawnerItemVariantComponentFix::fixSalmonBucket);
+               case "minecraft:axolotl_bucket" -> input.updateTyped(componentsFinder, (Fixer) EntitySpawnerItemVariantComponentFix::fixAxolotlBucket);
+               case "minecraft:tropical_fish_bucket" -> input.updateTyped(componentsFinder, (Fixer) EntitySpawnerItemVariantComponentFix::fixTropicalFishBucket);
                case "minecraft:painting" -> input.updateTyped(
                   componentsFinder,
                   components -> Util.writeAndReadTypedOrThrow(components, components.getType(), EntitySpawnerItemVariantComponentFix::fixPainting)
