From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/OminousBannerRarityFix.java b/net/minecraft/util/datafix/fixes/OminousBannerRarityFix.java
index bc749c828e1cafe5debd7f61507be5c79f1af3e6..8693f742bc3848bde76dce17b8909253cd680586 100644
--- a/net/minecraft/util/datafix/fixes/OminousBannerRarityFix.java
+++ b/net/minecraft/util/datafix/fixes/OminousBannerRarityFix.java
@@ -27,7 +27,7 @@ public class OminousBannerRarityFix extends DataFix {
       OpticFinder<?> blockEntityComponentsFieldFinder = blockEntityType.findField("components");
       OpticFinder<?> itemStackComponentsFieldFinder = itemStackType.findField("components");
       OpticFinder<?> itemNameFinder = blockEntityComponentsFieldFinder.type().findField("minecraft:item_name");
-      OpticFinder<Pair<String, String>> textComponentFinder = DSL.typeFinder(this.getInputSchema().getType(References.TEXT_COMPONENT));
+      OpticFinder<Pair<String, String>> textComponentFinder = (OpticFinder<Pair<String, String>>) DSL.typeFinder(this.getInputSchema().getType(References.TEXT_COMPONENT));
       return TypeRewriteRule.seq(this.fixTypeEverywhereTyped("Ominous Banner block entity common rarity to uncommon rarity fix", blockEntityType, input -> {
          Object blockEntityId = ((Pair)input.get(blockEntityIdFinder.finder())).getFirst();
          return blockEntityId.equals("minecraft:banner") ? this.fix(input, blockEntityComponentsFieldFinder, itemNameFinder, textComponentFinder) : input;
