From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/OverreachingTickFix.java b/net/minecraft/util/datafix/fixes/OverreachingTickFix.java
index b6c27e1b8e4dd2ac2d9a543aa716211485972824..ff5d77072097f0f0f4bdef298bc7468407f638ea 100644
--- a/net/minecraft/util/datafix/fixes/OverreachingTickFix.java
+++ b/net/minecraft/util/datafix/fixes/OverreachingTickFix.java
@@ -36,7 +36,7 @@ public class OverreachingTickFix extends DataFix {
       Dynamic<?> remainder, final int chunkX, final int chunkZ, final Optional<? extends Dynamic<?>> ticks, final String nameInUpgradeData
    ) {
       if (ticks.isPresent()) {
-         List<? extends Dynamic<?>> overreachingTicks = ((Dynamic)ticks.get()).asStream().filter(tick -> {
+         List<? extends Dynamic<?>> overreachingTicks = (ticks.get()).asStream().filter(tick -> {
             int x = tick.get("x").asInt(0);
             int z = tick.get("z").asInt(0);
             int distX = Math.abs(chunkX - (x >> 4));
