From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/RecipesFix.java b/net/minecraft/util/datafix/fixes/RecipesFix.java
index cd205430732a676361619f3352884a7395eb40c2..883299b20bdc0fc27280f0aea29c925e5b9ae5ac 100644
--- a/net/minecraft/util/datafix/fixes/RecipesFix.java
+++ b/net/minecraft/util/datafix/fixes/RecipesFix.java
@@ -4,7 +4,7 @@ import com.google.common.collect.ImmutableMap;
 import java.util.Map;
 
 public class RecipesFix {
-   public static final Map<String, String> RECIPES = ImmutableMap.builder()
+   public static final Map<String, String> RECIPES = ImmutableMap.<String, String>builder()
       .put("minecraft:acacia_wooden_slab", "minecraft:acacia_slab")
       .put("minecraft:birch_wooden_slab", "minecraft:birch_slab")
       .put("minecraft:black_stained_hardened_clay", "minecraft:black_terracotta")
