From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/datafix/fixes/StonecutterRecipeRenameFix.java b/net/minecraft/util/datafix/fixes/StonecutterRecipeRenameFix.java
index b3b4cd31607d6505d2841e381951d09312774284..d30ded693dc945c7d8e04cbd080550d53e19b51d 100644
--- a/net/minecraft/util/datafix/fixes/StonecutterRecipeRenameFix.java
+++ b/net/minecraft/util/datafix/fixes/StonecutterRecipeRenameFix.java
@@ -3,7 +3,7 @@ package net.minecraft.util.datafix.fixes;
 import com.google.common.collect.ImmutableMap;
 
 public class StonecutterRecipeRenameFix {
-   public static final ImmutableMap<String, String> RECIPE_RENAMES = ImmutableMap.builder()
+   public static final ImmutableMap<String, String> RECIPE_RENAMES = ImmutableMap.<String, String>builder()
       .put("minecraft:chiseled_stone_bricks_stone_from_stonecutting", "minecraft:chiseled_stone_bricks_from_stone_stonecutting")
       .put("minecraft:end_stone_brick_slab_from_end_stone_brick_stonecutting", "minecraft:end_stone_brick_slab_from_end_stone_bricks_stonecutting")
       .put("minecraft:end_stone_brick_stairs_from_end_stone_brick_stonecutting", "minecraft:end_stone_brick_stairs_from_end_stone_bricks_stonecutting")
@@ -16,7 +16,7 @@ public class StonecutterRecipeRenameFix {
       .put("minecraft:quartz_slab_from_stonecutting", "minecraft:quartz_slab_from_quartz_block_stonecutting")
       .put("minecraft:stone_brick_walls_from_stone_stonecutting", "minecraft:stone_brick_wall_from_stone_stonecutting")
       .build();
-   public static final ImmutableMap<String, String> ADVANCEMENT_RENAMES = ImmutableMap.builder()
+   public static final ImmutableMap<String, String> ADVANCEMENT_RENAMES = ImmutableMap.<String, String>builder()
       .put(
          "minecraft:recipes/building_blocks/chiseled_stone_bricks_stone_from_stonecutting",
          "minecraft:recipes/building_blocks/chiseled_stone_bricks_from_stone_stonecutting"
