From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/CubicSpline.java b/net/minecraft/util/CubicSpline.java
index 1d8d2f064bde0094901a3275d3ad97bcb0c66be2..95a5bc41f1e2a24ada0182c385db22770d623a83 100644
--- a/net/minecraft/util/CubicSpline.java
+++ b/net/minecraft/util/CubicSpline.java
@@ -32,18 +32,18 @@ public interface CubicSpline<C, I extends BoundedFloatFunction<C>> extends Bound
                Codec.lazyInitialized(result).fieldOf("value").forGetter(Point::value),
                Codec.FLOAT.fieldOf("derivative").forGetter(Point::derivative)
             )
-            .apply(i, (x$0, x$1, x$2) -> new Point(x$0, x$1, x$2))
+            .apply(i, (x$0, x$1, x$2) -> new Point<>(x$0, x$1, x$2))
       );
       Codec<CubicSpline.Multipoint<C, I>> multipointCodec = RecordCodecBuilder.create(
          i -> i.group(
                coordinateCodec.fieldOf("coordinate").forGetter(CubicSpline.Multipoint::coordinate),
                ExtraCodecs.nonEmptyList(pointCodec.listOf())
                   .fieldOf("points")
-                  .forGetter(
-                     m -> IntStream.range(0, m.locations.length)
-                        .mapToObj(p -> new Point(m.locations()[p], (CubicSpline<C, I>)m.values().get(p), m.derivatives()[p]))
+                       .forGetter(
+                m -> IntStream.range(0, m.locations.length)
+                        .mapToObj(p -> new Point<>(m.locations()[p], m.values().get(p), m.derivatives()[p]))
                         .toList()
-                  )
+                       )
             )
             .apply(i, (coordinate, points) -> {
                float[] locations = new float[points.size()];
