From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/PlaceholderLookupProvider.java b/net/minecraft/util/PlaceholderLookupProvider.java
index 984050268478b81b6f19c0290762774fc8669fcc..bf5937dfa1e704c0d00f5e4eda9bebf221ef364a 100644
--- a/net/minecraft/util/PlaceholderLookupProvider.java
+++ b/net/minecraft/util/PlaceholderLookupProvider.java
@@ -49,7 +49,7 @@ public class PlaceholderLookupProvider implements HolderGetter.Provider {
                   .map(RegistryOps.RegistryInfo::fromRegistryLookup)
                   .or(
                      () -> Optional.of(
-                        new RegistryOps.RegistryInfo(
+                        new RegistryOps.RegistryInfo<>(
                            PlaceholderLookupProvider.this.lookup.castAsOwner(), PlaceholderLookupProvider.this.lookup.castAsLookup(), Lifecycle.experimental()
                         )
                      )
@@ -112,11 +112,11 @@ public class PlaceholderLookupProvider implements HolderGetter.Provider {
       }
 
       public <T> HolderGetter<T> castAsLookup() {
-         return this;
+         return (HolderGetter<T>) this;
       }
 
       public <T> HolderOwner<T> castAsOwner() {
-         return this;
+         return (HolderOwner<T>) this;
       }
    }
 }
