From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/Util.java b/net/minecraft/util/Util.java
index 8c9233ce2f8b690b12d21c6474e4c2d44e95e527..69ee95019574018cf5bf3707d576204ceffb5e63 100644
--- a/net/minecraft/util/Util.java
+++ b/net/minecraft/util/Util.java
@@ -612,7 +612,7 @@ public class Util {
    public static <K extends Enum<K>, V> Map<K, V> makeEnumMap(final Class<K> keyType, final Function<K, V> function) {
       EnumMap<K, V> map = new EnumMap(keyType);
 
-      for (K key : (Enum[])keyType.getEnumConstants()) {
+      for (K key : keyType.getEnumConstants()) {
          map.put(key, function.apply(key));
       }
 
@@ -667,7 +667,7 @@ public class Util {
 
       for (int i = 0; i < futures.size(); i++) {
          int index = i;
-         decoratedFutures[i] = ((CompletableFuture)futures.get(i)).whenComplete((result, exception) -> {
+         decoratedFutures[i] = (futures.get(i)).whenComplete((result, exception) -> {
             if (exception != null) {
                failureHandler.accept(exception);
             } else {
@@ -1132,19 +1132,19 @@ public class Util {
    }
 
    public static <T> List<T> copyAndAdd(final List<T> list, final T element) {
-      return ImmutableList.builderWithExpectedSize(list.size() + 1).addAll(list).add(element).build();
+      return ImmutableList.<T>builderWithExpectedSize(list.size() + 1).addAll(list).add(element).build();
    }
 
    public static <T> List<T> copyAndAdd(final List<T> list, final T... elements) {
-      return ImmutableList.builderWithExpectedSize(list.size() + elements.length).addAll(list).add(elements).build();
+      return ImmutableList.<T>builderWithExpectedSize(list.size() + elements.length).addAll(list).add(elements).build();
    }
 
    public static <T> List<T> copyAndAdd(final T element, final List<T> list) {
-      return ImmutableList.builderWithExpectedSize(list.size() + 1).add(element).addAll(list).build();
+      return ImmutableList.<T>builderWithExpectedSize(list.size() + 1).add(element).addAll(list).build();
    }
 
    public static <K, V> Map<K, V> copyAndPut(final Map<K, V> map, final K key, final V value) {
-      return ImmutableMap.builderWithExpectedSize(map.size() + 1).putAll(map).put(key, value).buildKeepingLast();
+      return ImmutableMap.<K, V>builderWithExpectedSize(map.size() + 1).putAll(map).put(key, value).buildKeepingLast();
    }
 
    public static enum OS {
