From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/util/profiling/jfr/parse/JfrStatsResult.java b/net/minecraft/util/profiling/jfr/parse/JfrStatsResult.java
index 7d9119e2c908d45b3d6aa754cd7fb15daaef7315..6ac26a5f0e2835d197191cb0558aec120dc0de73 100644
--- a/net/minecraft/util/profiling/jfr/parse/JfrStatsResult.java
+++ b/net/minecraft/util/profiling/jfr/parse/JfrStatsResult.java
@@ -44,16 +44,16 @@ public record JfrStatsResult(
    List<StructureGenStat> structureGenStats
 ) {
    public List<Pair<ChunkStatus, TimedStatSummary<ChunkGenStat>>> chunkGenSummary() {
-      Map<ChunkStatus, List<ChunkGenStat>> byStatus = (Map<ChunkStatus, List<ChunkGenStat>>)this.chunkGenStats
+      Map<ChunkStatus, List<ChunkGenStat>> byStatus = this.chunkGenStats
          .stream()
          .collect(Collectors.groupingBy(ChunkGenStat::status));
       return byStatus.entrySet()
-         .stream()
-         .map(e -> Pair.of((ChunkStatus)e.getKey(), TimedStatSummary.summary((List)e.getValue())))
-         .filter(pair -> ((Optional)pair.getSecond()).isPresent())
-         .map(e -> Pair.of((ChunkStatus)e.getFirst(), (TimedStatSummary)((Optional)e.getSecond()).get()))
-         .sorted(Comparator.comparing(pair -> ((TimedStatSummary)pair.getSecond()).totalDuration()).reversed())
-         .toList();
+              .stream()
+              .map(e -> Pair.of(e.getKey(), TimedStatSummary.summary(e.getValue())))
+              .filter(pair -> pair.getSecond().isPresent())
+              .map(e -> Pair.of(e.getFirst(), e.getSecond().get()))
+              .sorted(Comparator.<Pair<ChunkStatus, TimedStatSummary<ChunkGenStat>>, Duration>comparing(pair -> pair.getSecond().totalDuration()).reversed())
+              .toList();
    }
 
    public String asJson() {
