From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/attribute/modifier/ColorModifier.java b/net/minecraft/world/attribute/modifier/ColorModifier.java
index 100ac7a036308063d7ed034cb041aab3f17cee0a..c19dc23cc6631edc54fabddb33dba96974e84041 100644
--- a/net/minecraft/world/attribute/modifier/ColorModifier.java
+++ b/net/minecraft/world/attribute/modifier/ColorModifier.java
@@ -25,10 +25,10 @@ public interface ColorModifier<Argument> extends AttributeModifier<Integer, Argu
          return LerpFunction.ofColor();
       }
    };
-   ColorModifier<Integer> ADD = ARGB::addRgb;
-   ColorModifier<Integer> SUBTRACT = ARGB::subtractRgb;
-   ColorModifier<Integer> MULTIPLY_RGB = ARGB::multiply;
-   ColorModifier<Integer> MULTIPLY_ARGB = ARGB::multiply;
+   ColorModifier<Integer> ADD = (RgbModifier) ARGB::addRgb;
+   ColorModifier<Integer> SUBTRACT = (RgbModifier) ARGB::subtractRgb;
+   ColorModifier<Integer> MULTIPLY_RGB = (RgbModifier) ARGB::multiply;
+   ColorModifier<Integer> MULTIPLY_ARGB = (RgbModifier) ARGB::multiply;
    ColorModifier<ColorModifier.BlendToGray> BLEND_TO_GRAY = new ColorModifier<ColorModifier.BlendToGray>() {
       public Integer apply(final Integer subject, final ColorModifier.BlendToGray argument) {
          int multipliedGreyscale = ARGB.scaleRGB(ARGB.greyscale(subject), argument.brightness);
