From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/advancements/criterion/EnterBlockTrigger.java b/net/minecraft/advancements/criterion/EnterBlockTrigger.java
index 85f7428f9e01162938ec9dfe36a017584f5355a3..de857a5b7808f4e51578758646210d18075c622d 100644
--- a/net/minecraft/advancements/criterion/EnterBlockTrigger.java
+++ b/net/minecraft/advancements/criterion/EnterBlockTrigger.java
@@ -24,7 +24,7 @@ public class EnterBlockTrigger extends SimpleCriterionTrigger<EnterBlockTrigger.
 
    public record TriggerInstance(Optional<ContextAwarePredicate> player, Optional<Holder<Block>> block, Optional<StatePropertiesPredicate> state)
       implements SimpleCriterionTrigger.SimpleInstance {
-      public static final Codec<EnterBlockTrigger.TriggerInstance> CODEC = RecordCodecBuilder.create(
+      public static final Codec<EnterBlockTrigger.TriggerInstance> CODEC = RecordCodecBuilder.<TriggerInstance>create(
             i -> i.group(
                   EntityPredicate.ADVANCEMENT_CODEC.optionalFieldOf("player").forGetter(EnterBlockTrigger.TriggerInstance::player),
                   BuiltInRegistries.BLOCK.holderByNameCodec().optionalFieldOf("block").forGetter(EnterBlockTrigger.TriggerInstance::block),
@@ -35,10 +35,10 @@ public class EnterBlockTrigger extends SimpleCriterionTrigger<EnterBlockTrigger.
          .validate(EnterBlockTrigger.TriggerInstance::validate);
 
       private static DataResult<EnterBlockTrigger.TriggerInstance> validate(final EnterBlockTrigger.TriggerInstance trigger) {
-         return (DataResult<EnterBlockTrigger.TriggerInstance>)trigger.block
-            .flatMap(
+         return trigger.block
+                 .<DataResult<EnterBlockTrigger.TriggerInstance>>flatMap(
                block -> trigger.state
-                  .flatMap(state -> state.checkState(((Block)block.value()).getStateDefinition()))
+                  .<String>flatMap(state -> state.checkState(((Block)block.value()).getStateDefinition()))
                   .map(property -> DataResult.error(() -> "Block" + block + " has no property " + property))
             )
             .orElseGet(() -> DataResult.success(trigger));
