From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/data/recipes/RecipeProvider.java b/net/minecraft/data/recipes/RecipeProvider.java
index b575f1e39b7ec8f6181c6fc5957a319ff56d6c55..074c79844b0ee1fbeec4295bab6769c0a7ed5ed3 100644
--- a/net/minecraft/data/recipes/RecipeProvider.java
+++ b/net/minecraft/data/recipes/RecipeProvider.java
@@ -63,7 +63,7 @@ public abstract class RecipeProvider {
    protected final HolderLookup.Provider registries;
    private final HolderGetter<Item> items;
    protected final RecipeOutput output;
-   private static final Map<BlockFamily.Variant, RecipeProvider.FamilyCraftingRecipeProvider> SHAPE_BUILDERS = ImmutableMap.builder()
+   private static final Map<BlockFamily.Variant, RecipeProvider.FamilyCraftingRecipeProvider> SHAPE_BUILDERS = ImmutableMap.<BlockFamily.Variant, RecipeProvider.FamilyCraftingRecipeProvider>builder()
       .put(
          BlockFamily.Variant.BUTTON, (RecipeProvider.FamilyCraftingRecipeProvider)(context, result, base) -> context.buttonBuilder(result, Ingredient.of(base))
       )
@@ -134,7 +134,7 @@ public abstract class RecipeProvider {
          )
       )
       .build();
-   private static final Map<BlockFamily.Variant, RecipeProvider.FamilyStonecutterRecipeProvider> STONECUTTER_RECIPE_BUILDERS = ImmutableMap.builder()
+   private static final Map<BlockFamily.Variant, RecipeProvider.FamilyStonecutterRecipeProvider> STONECUTTER_RECIPE_BUILDERS = ImmutableMap.<BlockFamily.Variant, RecipeProvider.FamilyStonecutterRecipeProvider>builder()
       .put(
          BlockFamily.Variant.SLAB,
          (RecipeProvider.FamilyStonecutterRecipeProvider)(context, result, base) -> context.stonecutterResultFromBase(
@@ -722,7 +722,7 @@ public abstract class RecipeProvider {
    }
 
    protected void waxRecipes(final FeatureFlagSet flagSet) {
-      ((BiMap)HoneycombItem.WAXABLES.get())
+      (HoneycombItem.WAXABLES.get())
          .forEach(
             (block, waxedBlock) -> {
                if (waxedBlock.requiredFeatures().isSubsetOf(flagSet)) {
