From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/storage/loot/functions/EnchantRandomlyFunction.java b/net/minecraft/world/level/storage/loot/functions/EnchantRandomlyFunction.java
index 757e2829793ab232570e02c599f4f625fb276803..c5cce6628cdf1165bcbc9ca91138f8a540452f1c 100644
--- a/net/minecraft/world/level/storage/loot/functions/EnchantRandomlyFunction.java
+++ b/net/minecraft/world/level/storage/loot/functions/EnchantRandomlyFunction.java
@@ -72,10 +72,10 @@ public class EnchantRandomlyFunction extends LootItemConditionalFunction {
       RandomSource random = context.getRandom();
       boolean targetIsBook = itemStack.is(Items.BOOK);
       boolean shouldCheckCompatibility = !targetIsBook && this.onlyCompatible;
-      Stream<Holder<Enchantment>> compatibleEnchantmentsStream = ((Stream)this.options
+      Stream<Holder<Enchantment>> compatibleEnchantmentsStream = (this.options
             .map(HolderSet::stream)
             .orElseGet(() -> context.getLevel().registryAccess().lookupOrThrow(Registries.ENCHANTMENT).listElements().map(Function.identity())))
-         .filter(candidate -> !shouldCheckCompatibility || ((Enchantment)candidate.value()).canEnchant(itemStack));
+         .filter(candidate -> !shouldCheckCompatibility || (candidate.value()).canEnchant(itemStack));
       List<Holder<Enchantment>> compatibleEnchantments = compatibleEnchantmentsStream.toList();
       Optional<Holder<Enchantment>> enchantment = Util.getRandomSafe(compatibleEnchantments, random);
       if (enchantment.isEmpty()) {
