From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/storage/loot/functions/ListOperation.java b/net/minecraft/world/level/storage/loot/functions/ListOperation.java
index 9b8c5da278f6e53fb004873a89d300e1ce3f6d27..a3177b05bac50d1c73f0d9dd51a4cd87f0df6ed1 100644
--- a/net/minecraft/world/level/storage/loot/functions/ListOperation.java
+++ b/net/minecraft/world/level/storage/loot/functions/ListOperation.java
@@ -18,7 +18,7 @@ public interface ListOperation {
    MapCodec<ListOperation> UNLIMITED_CODEC = codec(Integer.MAX_VALUE);
 
    static MapCodec<ListOperation> codec(final int maxSize) {
-      return ListOperation.Type.CODEC.dispatchMap("mode", ListOperation::mode, e -> e.mapCodec).validate(op -> {
+      return ListOperation.Type.CODEC.<ListOperation>dispatchMap("mode", ListOperation::mode, e -> e.mapCodec).validate(op -> {
          if (op instanceof ListOperation.ReplaceSection section && section.size().isPresent()) {
             int size = (Integer)section.size().get();
             if (size > maxSize) {
