From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/storage/loot/predicates/EnvironmentAttributeCheck.java b/net/minecraft/world/level/storage/loot/predicates/EnvironmentAttributeCheck.java
index eed5db5760c00f5c4b8d6b391c848117794c58c3..014670618c6b5b1588e31bf27e9a9faf9081b890 100644
--- a/net/minecraft/world/level/storage/loot/predicates/EnvironmentAttributeCheck.java
+++ b/net/minecraft/world/level/storage/loot/predicates/EnvironmentAttributeCheck.java
@@ -17,8 +17,8 @@ public record EnvironmentAttributeCheck<Value>(EnvironmentAttribute<Value> attri
    }
 
    @Override
-   public MapCodec<EnvironmentAttributeCheck<Value>> codec() {
-      return (MapCodec<EnvironmentAttributeCheck<Value>>)MAP_CODEC;
+   public MapCodec<EnvironmentAttributeCheck<?>> codec() {
+      return MAP_CODEC;
    }
 
    @Override
