From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/level/storage/loot/predicates/LootItemBlockStatePropertyCondition.java b/net/minecraft/world/level/storage/loot/predicates/LootItemBlockStatePropertyCondition.java
index f51a53c9002d3f552224774bb5ed1b33087151e0..a815e0d83a324c7ce162430a2807d0fd26577001 100644
--- a/net/minecraft/world/level/storage/loot/predicates/LootItemBlockStatePropertyCondition.java
+++ b/net/minecraft/world/level/storage/loot/predicates/LootItemBlockStatePropertyCondition.java
@@ -15,7 +15,7 @@ import net.minecraft.world.level.storage.loot.LootContext;
 import net.minecraft.world.level.storage.loot.parameters.LootContextParams;
 
 public record LootItemBlockStatePropertyCondition(Holder<Block> block, Optional<StatePropertiesPredicate> properties) implements LootItemCondition {
-   public static final MapCodec<LootItemBlockStatePropertyCondition> MAP_CODEC = RecordCodecBuilder.mapCodec(
+   public static final MapCodec<LootItemBlockStatePropertyCondition> MAP_CODEC = RecordCodecBuilder.<LootItemBlockStatePropertyCondition>mapCodec(
          i -> i.group(
                BuiltInRegistries.BLOCK.holderByNameCodec().fieldOf("block").forGetter(LootItemBlockStatePropertyCondition::block),
                StatePropertiesPredicate.CODEC.optionalFieldOf("properties").forGetter(LootItemBlockStatePropertyCondition::properties)
@@ -25,9 +25,9 @@ public record LootItemBlockStatePropertyCondition(Holder<Block> block, Optional<
       .validate(LootItemBlockStatePropertyCondition::validate);
 
    private static DataResult<LootItemBlockStatePropertyCondition> validate(final LootItemBlockStatePropertyCondition condition) {
-      return (DataResult<LootItemBlockStatePropertyCondition>)condition.properties()
+      return condition.properties()
          .flatMap(properties -> properties.checkState(condition.block().value().getStateDefinition()))
-         .map(name -> DataResult.error(() -> "Block " + condition.block() + " has no property" + name))
+         .map(name -> DataResult.<LootItemBlockStatePropertyCondition>error(() -> "Block " + condition.block() + " has no property" + name))
          .orElse(DataResult.success(condition));
    }
 
