From a80189141e4208c1ff8d8f37141f9c0d774a16d8 Mon Sep 17 00:00:00 2001
From: Oliver <oschi1303@gmail.com>
Date: Thu, 12 Mar 2026 16:39:22 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/animal/happyghast/HappyGhast.java b/net/minecraft/world/entity/animal/happyghast/HappyGhast.java
index 27967b4b275433195853aa5474b4ec6b09c54d69..b33276d993f3568cf179a60b937ec76e4003b3ce 100644
--- a/net/minecraft/world/entity/animal/happyghast/HappyGhast.java
+++ b/net/minecraft/world/entity/animal/happyghast/HappyGhast.java
@@ -385,7 +385,7 @@ public class HappyGhast extends Animal {
 
    @Override
    public Brain<HappyGhast> getBrain() {
-      return super.getBrain();
+      return (Brain<HappyGhast>) super.getBrain();
    }
 
    @Override
