From 4ae198aa20a8bea0d0555a1787f4d957ef55d2fb Mon Sep 17 00:00:00 2001
From: Strata <strata@fancyinnovations.com>
Date: Thu, 12 Mar 2026 18:33:15 +0100
Subject: [PATCH] Apply file patches


diff --git a/net/minecraft/world/entity/ai/behavior/GiveGiftToHero.java b/net/minecraft/world/entity/ai/behavior/GiveGiftToHero.java
index c12b4614275f57fa9db6c34871aa18f46a400555..05d88dc5dd209c7b939af98e98cff8d958c01c7f 100644
--- a/net/minecraft/world/entity/ai/behavior/GiveGiftToHero.java
+++ b/net/minecraft/world/entity/ai/behavior/GiveGiftToHero.java
@@ -21,7 +21,7 @@ public class GiveGiftToHero extends Behavior<Villager> {
    private static final int MIN_TIME_BETWEEN_GIFTS = 600;
    private static final int MAX_TIME_BETWEEN_GIFTS = 6600;
    private static final int TIME_TO_DELAY_FOR_HEAD_TO_FINISH_TURNING = 20;
-   private static final Map<ResourceKey<VillagerProfession>, ResourceKey<LootTable>> GIFTS = ImmutableMap.builder()
+   private static final Map<ResourceKey<VillagerProfession>, ResourceKey<LootTable>> GIFTS = ImmutableMap.<ResourceKey<VillagerProfession>, ResourceKey<LootTable>>builder()
       .put(VillagerProfession.ARMORER, BuiltInLootTables.ARMORER_GIFT)
       .put(VillagerProfession.BUTCHER, BuiltInLootTables.BUTCHER_GIFT)
       .put(VillagerProfession.CARTOGRAPHER, BuiltInLootTables.CARTOGRAPHER_GIFT)
