--- a/net/minecraft/world/entity/ai/behavior/InteractWith.java
+++ b/net/minecraft/world/entity/ai/behavior/InteractWith.java
@@ -30,7 +_,7 @@
       final int stopDistance
    ) {
       int interactionRangeSqr = interactionRange * interactionRange;
-      Predicate<LivingEntity> isTargetValid = mob -> mob.is(type) && targetFilter.test(mob);
+      Predicate<LivingEntity> isTargetValid = mob -> mob.is(type) && targetFilter.test((T)mob);
       return BehaviorBuilder.create(
          i -> i.group(
                i.registered(interactionTarget),
@@ -43,7 +_,7 @@
                if (selfFilter.test(body) && entities.contains(isTargetValid)) {
                   Optional<LivingEntity> closest = entities.findClosest(mob -> mob.distanceToSqr(body) <= interactionRangeSqr && isTargetValid.test(mob));
                   closest.ifPresent(mob -> {
-                     target.set(mob);
+                     target.set((T)mob);
                      lookTarget.set(new EntityTracker(mob, true));
                      walkTarget.set(new WalkTarget(new EntityTracker(mob, false), speedModifier, stopDistance));
                   });
